{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import json"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "with open('textList.json') as json_file:\n",
    "    d = json.load(json_file)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "dict_keys(['2010', '2011', '2012', '2013', '2014', '2015', '2016', '2017', '2018'])"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "d.keys()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "dict_keys(['Jan 05', 'Jan 19', 'Jan 26', 'Feb 02', 'Feb 09', 'Feb 16', 'Feb 23', 'Mar 02', 'Mar 09', 'Mar 16', 'Mar 23', 'Mar 30', 'Apr 06', 'Apr 13', 'Apr 20', 'Apr 27', 'May 04', 'May 11', 'May 18', 'May 25', 'Jun 01', 'Jun 08', 'Jun 15', 'Jun 22', 'Jun 29', 'Jul 06', 'Jul 13', 'Jul 20', 'Jul 27', 'Aug 03', 'Aug 10', 'Aug 17', 'Aug 24', 'Aug 31', 'Sep 07', 'Sep 14', 'Sep 21', 'Sep 28', 'Oct 05', 'Jan 12'])"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "d['2018'].keys()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'author': [],\n",
       "  'authorInstitution': [],\n",
       "  'category': 'This Week in\\n\\t\\t\\t\\t\\t\\t\\t\\tScience',\n",
       "  'date': '2018/01/12',\n",
       "  'doi': None,\n",
       "  'imageLink': None,\n",
       "  'pdfLink': 'http://science.sciencemag.org/content/359/6372/twis.full.pdf',\n",
       "  'pptLink': None,\n",
       "  'publisher': 'American Association for the Advancement of Science',\n",
       "  'text': \"Egyptian fruit bats were used to identify a brain region that recognizes fellow individuals. Different sets of neurons encode the spatial position and orientation of an organism. However, social animals need to know the position of other individuals for social interactions, observational learning, and group navigation. Surprisingly, very little is known about how the position of other animals is represented in the brain. Danjo et al. and Omer et al. now report the discovery of a subgroup of neurons in hippocampal area CA1 that encodes the presence of conspecifics in rat and bat brains, respectively. Science, this issue p. 213, p. 218 Neutron scattering can be used to tease out the details of collective magnetic excitations that yield well-defined peaks in the data. In principle, it could also be used to look into single-electron band excitations, but collecting enough data to capture broad distributions of intensity is tricky. Goremychkin et al. used neutron spectrometers that could efficiently capture a large amount of data by rotating the sample, a crystal of the intermediate-valence compound CePd3 (see the Perspective by Georges). The measured dynamical magnetic susceptibility, in combination with detailed ab initio calculations, showed the formation of coherent f-electron bands at low temperatures. Science, this issue p. 186; see also p. 162 Some locations on Mars are known to have water ice just below the surface, but how much has remained unclear. Dundas et al. used data from two orbiting spacecraft to examine eight locations where erosion has occurred. This revealed cliffs composed mostly of water ice, which is slowly sublimating as it is exposed to the atmosphere. The ice sheets extend from just below the surface to a depth of 100 meters or more and appear to contain distinct layers, which could preserve a record of Mars' past climate. They might even be a useful source of water for future human exploration of the red planet. Science, this issue p. 199 Galapagos finches have driven hypotheses of how speciation occurs. Most commonly, it is assumed that natural selection separates species originating from a single population on the basis of variation in traits that confer advantages for survival and reproduction. Lamichhaney et al. document a case where cross-species hybridization established a reproductively isolated lineage, which demonstrates a process known as homoploid hybrid speciation in action (see the Perspective by Wagner). The authors used genetic markers and phenotypic analyses to create a pedigree that revealed how a cross-island migrant bred with a native species to form a self-perpetuating hybrid population that was reproductively isolated from both parental species. Science, this issue p. 224; see also p. 157 Maintaining a healthy balance of gut bacteria can promote good health. Leonardi et al. show that fungi can also interact with gut immune cells to maintain intestinal well-being. CX3CR1+ mononuclear phagocytes (MNPs) patrol the intestine and promote antifungal immunity. Genetic deletion of CX3CR1 in MNPs caused colitis-like symptoms in mice. CX3CR1 polymorphisms were detected in Crohn's disease patients that were unable to produce antibodies against multiple fungal species. Thus, commensal fungi may be as important as bacteria in maintaining gut health, and antifungal therapy could hold promise for treating intestinal inflammation. Science, this issue p. 232 Large subaerial volcanic eruptions are among the most dramatic and intensively studied geological events on our planet, but similar submarine eruptions are less well understood. Carey et al. describe the largest submarine eruption in the past century. In 2012, the Havre volcano erupted off northern New Zealand at ocean depths in excess of 900 meters, producing an enormous raft of pumice—a gaseous froth of silica-rich lava. Sci. Adv. 10.1126/sciadv.1701121 (2017). The kinase ATM coordinates the response to DNA damage. Lee et al. report that ATM also coordinates a response to oxidative stress that is independent of its response to DNA damage. Activation of ATM by oxidative stress promoted mitochondrial function and autophagy, thus enabling cell survival through metabolic changes and the clearance of toxic protein aggregates. Thus, the loss of ATM in the neurodegenerative disease ataxia telangiectasia may reflect wide-ranging cellular stresses beyond a defective DNA-damage response. Sci. Signal. 10, eaan5598 (2018). Many bacterial pathogens inject their hosts with virulence effectors delivered by specialist secretion machines. Vibrio cholerae has a type VI secretion system (T6SS) that can be loaded with protein toxins that target eukaryote host cells or kill competing bacteria. Zhao et al. discovered that mutant V. cholerae lacking a T6SS could not compete against Escherichia coli strains in the mouse gut. In contrast, intact V. cholerae readily gained a foothold in the gut of young mice, pumping up inflammatory immune responses and prompting more violent symptoms. Science, this issue p. 210 The diffusion limitations on gas storage and catalytic reaction of microporous materials can often be overcome if they are incorporated into a mesoporous structure with much larger pores. Shen et al. grew ordered arrays of microcrystals of the ZIF-8 metal-organic framework, in which zinc ions are bridged by 2-methylimidazole linkers, inside a porous polystyrene template. These materials showed higher reaction rates for the Knoevenagel reaction between benzaldehydes and malononitriles and better catalyst recyclability. Science, this issue p. 206 Interferon-α/β receptor (IFNAR)-deficient mice are highly susceptible to viruses, including Zika virus (ZIKV). Previous studies modeled ZIKV infection during pregnancy in mice by crossing Ifnar1−/− females to wild-type males, generating Ifnar1+/− fetuses that retain type I IFN responsiveness. Yockey et al. directly examined the role of fetal type I IFN signaling in protection in this context by crossing Ifnar1−/− females to Ifnar1+/− males. Although Ifnar1−/− fetuses had higher ZIKV titers than Ifnar1+/− fetuses, Ifnar1−/− fetuses survived longer. Furthermore, activation of type I IFN signaling in the placentas of Ifnar1+/− fetuses led to fetal hypoxia, demise, and resorption. Sci. Immunol. 3, eaao1680 (2018). Transient receptor potential melastatin (TRPM) ion channels constitute the largest TRP subfamily and are involved in many physiological processes. TRPM8 is the primary cold and menthol sensor, and TRPM4 is associated with cardiovascular disorders. Yin et al. and Autzen et al. shed light on the general architecture of the TRPM subfamily by solving the structures of TRPM8 and TRPM4, respectively (see the Perspective by Bae et al.). The three-layered architecture of the TRPM8 channel provides the framework for understanding the mechanisms of cold and menthol sensing. The two distinct closed states of TRPM4, with and without calcium, reveal a calcium-binding site and calcium-binding-induced conformational changes. Science, this issue p. 237, p. 228; see also p. 160 The epigenetic states and associated chromatin dynamics underlying the initiation and maintenance of memory and effector CD8+ T cells are poorly understood. Pace et al. found that mice lacking the histone H3 lysine 9 methyltransferase Suv39h1 had markedly reduced antigen-specific effector CD8+ T cell responses to Listeria monocytogenes infection (see the Perspective by Henning et al.). Instead, CD8+ T cells in these mice were enriched for genes associated with naïve and memory signatures and showed enhanced memory potential and increased survival capacity. Thus, Suv39h1 marks chromatin through H3K9me3 deposition and silences memory and stem cell programs during the terminal differentiation of effector CD8+ T cells. Science, this issue p. 177; see also p. 163 Crohn's disease (CD), an inflammatory bowel disease, has a relatively high prevalence in Ashkenazi Jewish populations. Hui et al. conducted genome-wide association analysis in 2066 CD patients and 3633 healthy control individuals of Ashkenazi Jewish ancestry and identified two functional variants in the LRRK2 gene. The LRRK2 gene has been linked previously to the development of Parkinson's disease (PD). The newly discovered LRRK2 variants conferred risk for CD (N2081D) or protection from CD (N551K and R1398H). Analysis of other variants within the LRRK2 locus in 24,570 individuals revealed similar genetic effects between CD and PD in both Ashkenazi Jewish and non-Jewish cohorts. The presence of shared LRRK2 alleles in CD and PD provides insight into disease mechanisms and potential treatments. Sci. Transl. Med. 10, eaai7795 (2018). Aromatic molecules such as polycyclic aromatic hydrocarbons (PAHs) are known to exist in the interstellar medium owing to their characteristic infrared emission features. However, the infrared emission only indicates the general class of molecule, and identifying which specific molecular species are present is difficult. McGuire et al. used radio astronomy to detect rotational transitions of benzonitrile emitted from a well-known nearby cloud of interstellar gas (see the Perspective by Joblin and Cernicharo). This molecule may be a precursor to more complex PAHs. The identification of benzonitrile sheds light on the composition of aromatic material within the interstellar medium—material that will eventually be incorporated into new stars and planets. Science, this issue p. 202; see also p. 156 Malaria is a deadly disease with no effective vaccine. Physicians thus depend on antimalarial drugs to save lives, but such compounds are often rendered ineffective when parasites evolve resistance. Cowell et al. systematically studied patterns of Plasmodium falciparum genome evolution by analyzing the sequences of clones that were resistant to diverse antimalarial compounds across the P. falciparum life cycle (see the Perspective by Carlton). The findings identify hitherto unrecognized drug targets and drug-resistance genes, as well as additional alleles in known drug-resistance genes. Science, this issue p. 191; see also p. 159 Many eukaryotic proteins are modified by the attachment of lipids, and these modifications can alter how proteins interact with cellular membranes. Rana et al. present x-ray crystal structures of an integral membrane enzyme that appends a fatty acyl chain onto a cysteine residue of target proteins. The enzyme active site is situated at the membrane surface, thus explaining the enzyme's preference for substrates that are already membrane-associated. The structure of a fatty acid-like inhibitor bound within a hydrophobic cavity elucidates the mechanism for the enzyme's acyl chain specificity. Science, this issue p. eaao6326 Nearly 50 years after the concept was first proposed, gene therapy is now considered a promising treatment option for several human diseases. The path to success has been long and tortuous. Serious adverse effects were encountered in early clinical studies, but this fueled basic research that led to safer and more efficient gene transfer vectors. Gene therapy in various forms has produced clinical benefits in patients with blindness, neuromuscular disease, hemophilia, immunodeficiencies, and cancer. Dunbar et al. review the pioneering work that led the gene therapy field to its current state, describe gene-editing technologies that are expected to play a major role in the field's future, and discuss practical challenges in getting these therapies to patients who need them. Science, this issue p. eaan4672 \",\n",
       "  'textLink': None,\n",
       "  'title': 'This Week in Science: Research in Science journals.',\n",
       "  'type': None},\n",
       " {'author': ['Richard Stone'],\n",
       "  'authorInstitution': ['Havana'],\n",
       "  'category': 'In Depth',\n",
       "  'date': '2018/01/12',\n",
       "  'doi': '10.1126/science.359.6372.144',\n",
       "  'imageLink': None,\n",
       "  'pdfLink': 'http://science.sciencemag.org/content/359/6372/144.full.pdf',\n",
       "  'pptLink': '/content/359/6372/144.full.txt',\n",
       "  'publisher': 'American Association for the Advancement of Science',\n",
       "  'text': \"Habaneros wade through floodwaters near El Malecón after Hurricane Irma. On its deadly run through the Caribbean last September, Hurricane Irma lashed northern Cuba, inundating coastal settlements and scouring away vegetation. The powerful storm dealt Havana only a glancing blow; even so, 10-meter waves pummeled El Malecón, the city's seaside promenade, and ravaged stately but decrepit buildings in the capital's historic district. “There was great destruction,” says Dalia Salabarría Fernández, a marine biologist here at the National Center for Protected Areas (CNAP). As the flood waters receded, she says, “Cuba learned a very important lesson.” With thousands of kilometers of low-lying coast and a location right in the path of Caribbean hurricanes, which many believe are intensifying because of climate change, the island nation must act fast to gird against future disasters. Irma lent new urgency to a plan, called Tarea Vida, or Project Life, adopted last spring by Cuba's Council of Ministers. A decade in the making, the program bans construction of new homes in threatened coastal areas, mandates relocating people from communities doomed by rising sea levels, calls for an overhaul of the country's agricultural system to shift crop production away from saltwater-contaminated areas, and spells out the need to shore up coastal defenses, including by restoring degraded habitat. “The overarching idea,” says Salabarría Fernández, “is to increase the resilience of vulnerable communities.” But the cash-strapped government had made little headway. Now, “Irma [has] indicated to everybody that we need to implement Tarea Vida in a much more rapid way,” says Orlando Rey Santos, head of the environment division at Cuba's Ministry of Science, Technology and Environment (CITMA) here, which is spearheading the project. The government aims to spend at least $40 million on Project Life this year, and it has approached overseas donors for help. Italy was the first to respond, pledging $3.4 million to the initiative in November 2017. A team of Cuban experts has just finished drafting a $100 million proposal that the government plans to submit early this year to the Global Climate Fund, an international financing mechanism set up under the United Nations Framework Convention on Climate Change. Many countries with vulnerable coastlines are contemplating similar measures, and another island nation—the Seychelles—has offered to collaborate on boosting coastal protection in Cuba. But Project Life stands out for taking a long view: It intends to prepare Cuba for climatological impacts over the next century. “It's impressive,” says marine scientist David Guggenheim, president of Ocean Doctor, a nonprofit in Washington, D.C., that has projects in Cuba. “Cuba is an unusual country in that they actually respect their scientists, and their climate change policy is science driven.” Rising sea levels pose the most daunting challenge for Cuba. Over the past half-century, CITMA says, average sea levels have risen some 7 centimeters, wiping out low-lying beaches and threatening marsh vegetation, especially along Cuba's southern midsection. The coastal erosion is “already much worse than anyone expected,” Salabarría Fernández says. Storms drive the rising seas farther inland, contaminating coastal aquifers and croplands. Still worse is in store, even in conservative scenarios of sea-level rise, which forecast an 85-centimeter increase by 2100. According to the latest CITMA forecast, seawater incursion will contaminate nearly 24,000 square kilometers of land this century. About 20% of that land could become submerged. “That means several percent of Cuban land will be underwater,” says Armando Rodríguez Batista, director of science, technology, and innovation at CITMA. To shore up the coastlines, Project Life aims to restore mangroves, which constitute about a quarter of Cuba's forest cover. “They are the first line of defense for coastal communities. But so many mangroves are dying now,” Salabarría Fernández says. Leaf loss from hurricane-force winds, erosion, spikes in salinity, and nutrient imbalances could all be driving the die-off, she says. Coral reefs can also buffer storms. A Cuban-U.S. expedition that circumnavigated the island last spring found that many reefs are in excellent health, says Juliett González Méndez, a marine ecologist with CNAP. But at a handful of hot spots, reefs exposed to industrial effluents are ailing, she says. One Project Life target is to squelch runoff and restore those reefs. Another pressing need is coastal engineering. Topping Cuba's wish list are jetties or other wave-disrupting structures for protecting not only the iconic Malecón, but also beaches and scores of tiny keys frequented by tourists whose spending is a lifeline for many Cubans. Cuba has appealed to the Netherlands to lend its expertise in coastal engineering. Perhaps the thorniest element of Project Life is a plan to relocate low-lying villages. As the sea invades, “some communities will disappear,” Salabarría Fernández says. The first relocations under the initiative took place in October 2017, when some 40 families in Palmarito, a fishing village in central Cuba, were moved inland. Other communities may not need to pull up stakes for decades. But Cuban social scientists are already fanning out to those ill-fated villages to educate people on climate change and win them over on the eventual need to move. That's an easier sell in the wake of a major hurricane, Rodríguez Batista says. “Irma has helped us with public awareness,” he says. “People understand that climate change is happening now.” Nation seeks assistance for project to strengthen coastal defenses and relocate villages. Nation seeks assistance for project to strengthen coastal defenses and relocate villages. \",\n",
       "  'textLink': '/content/359/6372/144.full.txt',\n",
       "  'title': \"Cuba's 100-year plan for climate change\",\n",
       "  'type': 'News'},\n",
       " {'author': ['Chanhyung Bae', 'Andres Jara-Oseguera', 'Kenton J. Swartz'],\n",
       "  'authorInstitution': ['Molecular Physiology and Biophysics Section, Porter Neuroscience Research Center, National Institute of Neurological Disorders and Stroke, National Institutes of Health, Bethesda, MD 20892, USA.',\n",
       "   'Molecular Physiology and Biophysics Section, Porter Neuroscience Research Center, National Institute of Neurological Disorders and Stroke, National Institutes of Health, Bethesda, MD 20892, USA.',\n",
       "   'Molecular Physiology and Biophysics Section, Porter Neuroscience Research Center, National Institute of Neurological Disorders and Stroke, National Institutes of Health, Bethesda, MD 20892, USA.'],\n",
       "  'category': 'Perspectives',\n",
       "  'date': '2018/01/12',\n",
       "  'doi': '10.1126/science.aar6205',\n",
       "  'imageLink': None,\n",
       "  'pdfLink': 'http://science.sciencemag.org/content/359/6372/160.full.pdf',\n",
       "  'pptLink': '/content/359/6372/160.full.txt',\n",
       "  'publisher': 'American Association for the Advancement of Science',\n",
       "  'text': \"Transient receptor potential (TRP) channels were first identified in photoreceptors of the fruit fly (1, 2). In mammals, six major families of TRP channels play key roles in sensing stimuli such as light, temperature, membrane lipids, and intracellular Ca2+. In 2013, two landmark publications revealed the cryo–electron microscopy (cryo-EM) structure of the heat- and capsaicin-activated TRPV1 channel (3, 4). Two articles in this issue report cryo-EM structures of cation-selective TRPM channels. On page 228, Autzen et al. (5) describe TRPM4, which is activated by intracellular Ca2+ and involved in controlling arterial tone, cardiac rhythm, and the immune response (6). On page 237, Yin et al. (7) report on TRPM8, which senses cold and menthol and may serve as a cancer biomarker (8). Autzen et al. solved structures of TRPM4 in the absence and presence of the activating ion Ca2+. The transmembrane domains of TRPM4 resemble those of other TRP channels and of voltage-activated K+ (Kv) channels (3, 4, 9). Each subunit in the tetrameric complex contains six transmembrane helices, with S5–S6 forming the ion permeation pathway at the central axis and S1–S4 forming a peripheral domain (see the figure). In the Ca2+-bound TRPM4 structure, the ion binds in a cavity within the S1–S4 domain that faces the cytoplasm (see the figure). Both the Ca2+-bound and -unbound TRPM4 structures have been captured in states in which the intracellular entrance of the pore is sealed off by hydrophobic residues in the S6 helix, similar to the closed structures of other TRP and Kv channels (3, 4, 9, 10). The ion selectivity filter within the external pore of TRPM4 contains a conserved trio of residues (Phe-Gly-Gln or FGQ) that appear to form cation-binding sites. Overall, the TRPM8 structure solved by Yin et al. is remarkably similar to that of TRPM4. Differences in regions that connect domains make it difficult to appreciate their relatedness when overlaying the complete structures, but superposition of individual domains shows that they adopt very similar folds (see the figure). The TRPM8 structure contains a similar cavity to that which binds Ca2+ in TRPM4; the channel is closed and the selectivity filter (not resolved in the structure) contains the conserved FGQ sequence. It is unclear why TRPM4 and its closest homolog, TRPM5, are selective for monovalent cations, whereas TRPM8 and all other TRP channels are also permeable to divalent cations. The TRPM8 pore may be wider than that in TRPM4, but higher-resolution structures will be needed to understand the unique ion selectivity mechanisms in these channels. Structure of the TRPM8 channel, with individual domains of TRPM4 superimposed to illustrate structural similarities. The enlarged view illustrates the calcium ion–binding cavity within the S1–S4 domain of TRPM4. The TRPM4 Ca2+ binding site is particularly fascinating because it suggests the presence of a polymodal ligand-binding cavity that is likely a conserved feature of TRPM channels. Comparison of the Ca2+-binding cavities of the Ca2+-bound and -unbound TRPM4 structures shows local conformational rearrangements that seem to propagate down to the TRP domain, a conserved region among TRP channels that is important for opening these channels. Ca2+ binding to the cavity in TRPM4 is required for channel opening, and Ca2+ is also required for TRPM8 activation by the synthetic compound icilin (11). Comparison of the TRPM4 and TRPM8 Ca2+-binding cavities shows that they are remarkably conserved, particularly in the absence of Ca2+; they are also conserved in the other Ca2+-modulated TRPM channels (TRPM2 and TRPM5). Residues required for activation of TRPM8 by menthol and icilin (11, 12) are located in the Ca2+-binding cavity, suggesting that this region is a polymodal ligand-binding site in different types of TRPM channels. However, most residues required for TRPM8 activation by menthol and icilin are also conserved in TRPM4, raising questions about why that channel is not sensitive to these compounds or why TRPM8 activation by menthol does not require Ca2+ binding. The TRPV1 channel has some overlapping pharmacology with TRPM8, and both TRPV1 and TRPV2 have a polymodal ligand-binding cavity in roughly the same area as TRPM8 (13). However, comparison of these cavities in TRPV1 and TRPM8 reveals little conservation, suggesting distinct ligand-binding mechanisms. The overall hydrophobicity of the S1–S4 helices for TRPM4 and TRPM8 channels, which are activated by positive membrane voltages, suggests a different voltage-sensing mechanism than in Kv channels; in the latter, Arg residues in the S4 helix drive movement of this helix through the membrane (9). The TRPM structures (5, 7) provide a first glimpse of their conserved and unique amino termini, which contain four melastatin homology regions (MHRs) in all TRPM channels. These MHRs derive their name from melastatin, a metastasis suppressor in melanoma cells that is a soluble splice variant of the amino terminus of TRPM1. The functions of these amino-terminal domains are poorly understood, but two other recently reported TRPM4 structures suggest that the MHRs can regulate the functional properties of the channel. In Guo et al.'s structure, the allosteric inhibitor adenosine 5′-triphosphate is bound at the interface between MHR1 and MHR2 of two neighboring subunits (14). In a second study, Winkler et al. identified two cavities lined by basic amino acids where the polyanionic decavanadate binds to prevent channel closure at negative membrane voltages (15). Comparison of all available TRPM4 structures (5, 14, 15) reveals only relatively subtle conformational changes; in each case, the pore remains closed. In the future, it will be critical to solve structures of these channels in an open state to understand the mechanism of channel opening and its regulation by ligands. That none of these structures captured an open state might be due to the absence of phosphatidylinositol 4,5-bisphosphate, which is required for opening of both TRPM4 and TRPM8, as well as the absence of membrane voltage. It may be worthwhile to search for mutants that stabilize the open state for TRPM channels. In the case of TRPM8, agonists such as menthol and icilin may help to capture an open state. These exciting new structures provide a framework for future studies. It will, for example, be interesting to further investigate how the MHR domains regulate channel function, and to explore possible roles in protein localization, trafficking, or protein-protein interactions. It would also be interesting to reexamine the origin of the voltage sensitivity of TRPM channels, a feature common to other TRP channels but whose mechanism is unclear. The new structures will also catalyze experiments to explore the mechanisms underlying temperature sensing in TRP channels, with the structure of TRPM8 providing a new foundation for investigating the mechanism by which this channel is activated by cold. This is an article distributed under the terms of the Science Journals Default License. The structures of TRPM channels help to explain how they can sense intracellular calcium The structures of TRPM channels help to explain how they can sense intracellular calcium \",\n",
       "  'textLink': '/content/359/6372/160.full.txt',\n",
       "  'title': 'TRPM channels come into focus',\n",
       "  'type': 'Article Commentary'},\n",
       " {'author': ['Mitra S. Rana',\n",
       "   'Pramod Kumar',\n",
       "   'Chul-Jin Lee',\n",
       "   'Raffaello Verardi',\n",
       "   'Kanagalaghatta R. Rajashankar',\n",
       "   'Anirban Banerjee'],\n",
       "  'authorInstitution': ['Cell Biology and Neurobiology Branch, National Institutes of Child Health and Human Development, National Institutes of Health, Bethesda, MD 20892, USA.',\n",
       "   'Cell Biology and Neurobiology Branch, National Institutes of Child Health and Human Development, National Institutes of Health, Bethesda, MD 20892, USA.',\n",
       "   'Cell Biology and Neurobiology Branch, National Institutes of Child Health and Human Development, National Institutes of Health, Bethesda, MD 20892, USA.',\n",
       "   'Cell Biology and Neurobiology Branch, National Institutes of Child Health and Human Development, National Institutes of Health, Bethesda, MD 20892, USA.',\n",
       "   'Northeastern Collaborative Access Team (NE-CAT) and Department of Chemistry and Chemical Biology, Cornell University, Building 436E, Argonne National Laboratory, 9700 South Cass Avenue, Argonne, IL 60439, USA.',\n",
       "   'Cell Biology and Neurobiology Branch, National Institutes of Child Health and Human Development, National Institutes of Health, Bethesda, MD 20892, USA.'],\n",
       "  'category': 'Research Articles',\n",
       "  'date': '2018/01/12',\n",
       "  'doi': '10.1126/science.aao6326',\n",
       "  'imageLink': None,\n",
       "  'pdfLink': 'http://science.sciencemag.org/content/359/6372/eaao6326.full.pdf',\n",
       "  'pptLink': '/content/359/6372/eaao6326.full.txt',\n",
       "  'publisher': 'American Association for the Advancement of Science',\n",
       "  'text': \"↵* These authors contributed equally to this work. Many eukaryotic proteins are modified by the attachment of lipids, and these modifications can alter how proteins interact with cellular membranes. Rana et al. present x-ray crystal structures of an integral membrane enzyme that appends a fatty acyl chain onto a cysteine residue of target proteins. The enzyme active site is situated at the membrane surface, thus explaining the enzyme's preference for substrates that are already membrane-associated. The structure of a fatty acid-like inhibitor bound within a hydrophobic cavity elucidates the mechanism for the enzyme's acyl chain specificity. Science, this issue p. eaao6326 Hundreds of cellular proteins are modified by posttranslational S-acylation of cysteines, commonly known as protein palmitoylation. Unlike other lipid attachments, which are thought to be permanent, palmitoylation can be reversed by cellular thioesterases, enabling dynamic modulation of the local hydrophobicity of substrate proteins. In humans, palmitoylation is catalyzed by 23 members of the DHHC family of integral membrane enzymes, which contain a signature Asp-His-His-Cys (DHHC) motif. DHHC enzymes use acyl–coenzyme A (CoA) (predominantly palmitoyl-CoA) to generate an acyl-enzyme intermediate from which the acyl chain is subsequently transferred to a substrate. With a recent systems-level analysis suggesting that more than 10% of the proteome is palmitoylated, the complexity of protein palmitoylation approaches that of protein phosphorylation and ubiquitylation. Nonetheless, fundamental aspects of DHHC enzymes, including their mechanism of catalysis and acyl-CoA binding and recognition, have been challenging to analyze without detailed structural information. To obtain insights into the structural mechanism of DHHC enzymes, we solved the crystal structures of two DHHC family members: human DHHC20 and a catalytically inactive mutant of zebrafish DHHC15. We also purified and solved the structure of human DHHC20 conjugated to an irreversible inhibitor that mimics the acylated enzyme intermediate. We carried out structure-guided mutagenesis experiments to test residues important for enzyme function and to engineer mutant enzymes with altered acyl-CoA selectivity. The four transmembrane helices of hDHHC20 and zfDHHS15 form a tepee-like structure with the active site, contained in the highly conserved cytosolic DHHC cysteine-rich domain, at the membrane-cytosol interface. The cysteine-rich domain binds two zinc (Zn2+) ions that impart structural stability, but do not actively coordinate the nucleophilic cysteine. The C-terminal domain has an unanticipated amphipathic helix and a hydrophobic loop that together form a supporting structure for the transmembrane domain. The transmembrane domain forms a cavity where the acyl chain of acyl-CoA is inserted. Cavity-lining residues are determinants of fatty acyl recognition and chain-length selectivity. Our structures enabled us to engineer mutants of human DHHC20 with altered acyl chain–length selectivities. By elucidating the location of the active site at the membrane-aqueous interface, our structures readily explain why candidate cysteines for palmitoylation are proximal to the membrane. The active site has a catalytic triad-like arrangement of aspartic acid and histidine residues that activate the nucleophilic cysteine. The C-terminal domain has conserved motifs that form critical interactions with the active site and the rest of the protein. The structures reported here set the stage for the development of structure-based small molecule probes and tools such as orthogonal DHHC enzyme–fatty acyl–CoA pairs that will likely help investigate the enzyme-substrate network of this biologically and biomedically important family of enzymes. Human DHHC20 palmitoyltransferase (yellow) shown localized in the Golgi body membrane (green stacks). The Zn2+ ions are shown as gray spheres and the acyl chain of the palmitoyl group in white sticks. A hypothetical substrate (blue) approaches the palmitoyltransferase from the left and, after palmitoylation, is localized to the Golgi body membrane through anchoring of the palmitoyl group, now transferred to the substrate. DHHC (Asp-His-His-Cys) palmitoyltransferases are eukaryotic integral membrane enzymes that catalyze protein palmitoylation, which is important in a range of physiological processes, including small guanosine triphosphatase (GTPase) signaling, cell adhesion, and neuronal receptor scaffolding. We present crystal structures of two DHHC palmitoyltransferases and a covalent intermediate mimic. The active site resides at the membrane-cytosol interface, which allows the enzyme to catalyze thioester-exchange chemistry by using fatty acyl–coenzyme A and explains why membrane-proximal cysteines are candidates for palmitoylation. The acyl chain binds in a cavity formed by the transmembrane domain. We propose a mechanism for acyl chain–length selectivity in DHHC enzymes on the basis of cavity mutants with preferences for shorter and longer acyl chains. Covalent attachment to lipids is a prevalent form of posttranslational modification (1) that influences membrane association of peripheral membrane proteins, protein targeting to membrane subdomains, protein-protein associations, and protein trafficking (2, 3). Of the main different forms of protein lipidation, attachment of a 14-carbon myristoyl group to the N terminus or a 15-carbon farnesyl or 20-carbon geranylgeranyl to the C terminus results in a relatively stable modification (4). By contrast, attachment of a fatty acyl group to an internal cysteine as a thioester (5, 6), known as protein S-acylation, is readily reversible through the action of cellular thioesterases (7, 8), making S-acylation a potentially dynamic form of lipidation (9). Protein S-acylation is more commonly referred to as protein palmitoylation owing to the prevalence of the 16-carbon palmitate among the acyl chains that are attached to substrate proteins (10). However, for at least a subset of proteins, modification by fatty acyl chains longer or shorter than 16 carbons has been shown (10–12). The readily reversible nature of protein palmitoylation enables dynamic modulation of the hydrophobicity of substrate proteins. Protein palmitoylation plays critical roles in a wide range of physiological processes such as Ras signaling (13), localization of neuronal scaffolding proteins (14), intracellular trafficking (15), activity of ion channels (16), and host-pathogen interactions (17, 18). Since their discovery, an increasing number of proteins have been added to the repertoire of cellular proteins that are palmitoylated, with a recent estimate of close to 1000 proteins in humans (19). Although bioinformatic analyses of protein sequences proximal to the target cysteine have had some success in predicting palmitoylation sites, there are currently no reported consensus sequences for palmitoylation (20). Examination of experimentally identified palmitoylation sites and their sequence context, both in terms of physicochemical properties as well as predicted structure, is strongly indicative of the fact that one of the criteria for a cysteine to be palmitoylated is proximity to the membrane (20). Protein palmitoylation is connected to diseases, especially cancers and neuropsychiatric disorders (21), and it has been proposed that developing inhibitors of DHHC20, an enzyme that palmitoylates epidermal growth factor receptor (EGFR), can provide a therapeutic avenue for treating cancers that are resistant to EGFR-targeted therapy (22). Although palmitoylation as a posttranslational modification was discovered in 1979 (5), the enzymes that catalyze protein palmitoylation were only discovered in 2002 (23, 24). These are low-abundance, polytopic eukaryotic integral membrane enzymes known as DHHC-palmitoyl transferases, so named because they contain a signature Asp-His-His-Cys (DHHC) motif within a cysteine-rich domain in an intracellular loop (fig. S1). Localization studies suggest that DHHC enzymes reside at a variety of cellular compartments, most prominently the Golgi complex (25). Beyond the shared cysteine-rich domain, there is considerable diversity in the DHHC family—some possess ankyrin repeats (24), a few have six transmembrane (TM) helices (26) instead of the canonical four, and at least one of them forms a functional heterodimer with an auxiliary subunit (23). Studies of yeast Erf2/Erf4 (homolog of mammalian DHHC9/GCP16) (27) and mammalian DHHC2 and DHHC3 (28) indicate that palmitate transfer to substrates occurs in two steps. First, autoacylation of a cysteine within the enzyme with palmitoyl–coenzyme A (CoA) forms a palmitoylated intermediate. This intermediate has been isolated in vitro, and, in the absence of a substrate, the autopalmitoylated enzyme undergoes slow hydrolysis. However, in presence of a protein substrate, the palmitate is transferred to a cysteine on the substrate in a transpalmitoylation reaction that regenerates the DHHC enzyme (28) (Fig. 1A). The specific roles of the conserved residues in the cysteine-rich domain that includes the DHHC motif are poorly understood. Genetic and biochemical analyses indicates that DHHC enzymes bind two Zn2+ ions (29) at two zinc finger–like domains, but the function of these Zn2+ ions in DHHC enzymes is unknown. Moreover, fatty acyl–CoA selectivity varies between DHHC enzymes (28, 30). Nevertheless, nothing is known about the site on the enzyme where the acyl-CoA binds and, thus, the determinants for fatty acid chain–length selectivity. (A) The proposed two-step catalytic mechanism of the DHHC enzymes in literature, where they first undergo self-acylation to form an acylated intermediate (shown in black) and subsequently transfer the acyl chain to a protein substrate in the second step (shown in blue). (B) Determination of kinetic parameters for the autopalmitoylation of hDHHC20 measured using the coupled-enzyme assay (fig. S3A). Shown is a Michaelis-Menten fit to palmitoyl-CoA titration with hDHHC20 (red circle) and hDHHS20 (black square). Km = 0.58 ± 0.04 μM; kcat = 16.5 ± 0.3 min−1. (C) Same as (B) for zfDHHC15. Km = 10 ± 1 μM; kcat = 14.6 ± 0.8 min−1. Data are mean ± SEM of three or more replicate measurements. We carried out an extensive search for appropriate constructs of members of the DHHC family for crystallization using fluorescence size-exclusion chromatography (FSEC), screening for protein stability, yield, and monodispersity of size-exclusion chromatographic profile (31). Of the initial hits, human DHHC20 (hDHHC20) and zebrafish DHHC15 (zfDHHC15) were promising candidates (figs. S2 and S4). Because there was no biochemical characterization of either DHHC20 or DHHC15 in the literature, we tested whether they are bona fide palmitoyltransferase enzymes. We first used a coupled-enzyme assay that interrogates DHHC autoacylation by utilizing the released free CoA to generate an equivalent amount of reduced nicotinamide adenine dinucleotide (NADH) (32) (Fig. 1A), which is detected by its fluorescence (fig. S3A). Both hDHHC20 and zfDHHC15 showed robust activity (Fig. 1, B and C, and figs. S3C and S4E). Mutants in which the active site cysteine was changed to a serine, designated as hDHHS20 and zfDHHS15, showed very little activity. To demonstrate the capability of the enzymes for catalyzing palmitoylation of protein substrates, we used the Legionella effector protein GobX (17) and human SNAP25b (30) as substrates for hDHHC20 and zfDHHC15, respectively. In an in vitro assay with purified proteins, hDHHC20 and zfDHHC15 displayed robust palmitoyl-transfer activity, whereas the catalytically inactive mutants, hDHHS20 and zfDHHS15, displayed the same level of activity as the “no enzyme” control (fig. S5). These experiments demonstrated that our preparations of hDHHC20 and zfDHHC15 have robust autoacylation activity in a fully reconstituted in vitro assay and are competent for substrate palmitoylation. Purified hDHHC20 and zfDHHC15 contain two Zn2+ ions per protein (figs. S2D and S4F), similar to what has been observed with DHHC3 (33) and Swf1, a yeast DHHC enzyme (29). We obtained crystals of wild-type hDHHC20 from lipidic cubic phase (LCP) and crystals of the mutant, zfDHHS15, from hanging drop conditions. These crystals yielded diffraction data that enabled us to solve the structure using Zn–single-wavelength anomalous dispersion (SAD) data from the Zn2+ ions in the protein (hDHHC20) and with a combination of Zn-SAD and sulfur-SAD for zfDHHS15 (fig. S6, F to I). The hDHHC20 protein crystallizes in two different crystal forms in LCP: a P63 form with an antiparallel dimer and a P21 form with a loose parallel dimer in the asymmetric unit (fig. S6, A and B). Conversely, zfDHHS15 crystallizes in P42212 space group with an antiparallel dimer in the asymmetric unit (fig. S6C). We obtained x-ray diffraction data of zfDHHS15 crystals to 2.54-Å resolution. The hDHHC20 protein was reductively methylated for obtaining optimal diffraction data. The hexagonal form of hDHHC20 diffracted to 2.44-Å resolution. The DHHC structures display similar monomer structures, with a root mean square deviation (RMSD) of superposition (Cα) between the monomers of the two forms of hDHHC20 being 0.52 Å and between the monomer of zfDHHS15 and the monomer of hDHHC20 (P63 form) being 0.74 Å (fig. S6, D and E). We will focus our discussion on the hexagonal form of hDHHC20, with comments on zfDHHS15 where appropriate. The final models contain residues 5 to 299 of hDHHC20 and residues 8 to 298 of zfDHHS15 (residues 42 and 43 and 199 to 202 are absent from the zfDHHS15 model) (tables S1 and S2). Protein palmitoylation almost exclusively occurs at the cytoplasmic face of organellar and plasma membranes, and thus the predicted topology of DHHC enzymes would place the DHHC loop in the cytoplasm. Based on this and the fact that the same hDHHC20 protein crystallized in lipid-rich phase in two different forms, we interpret the different dimeric interfaces as adventitious protein-protein interactions. Such dimer formation has been seen in other membrane-protein structures (34, 35). To rule out hDHHC20 or zfDHHC15 adopting an inverted topology, we carried out a fluorescence protease protection assay (36) to determine the physiological topology of both hDHHC20 and zfDHHC15. Both hDHHC20 and zfDHHC15 are putative Golgi resident proteins, and C-terminal green fluorescent protein (GFP)–tagged constructs localize to the Golgi. Upon permeabilization of the plasma membrane with digitonin and addition of trypsin, the fluorescent tag on the C terminus is not protected, whereas a control with a fluorescent tag in the lumen of the Golgi is (figs. S7 and S8). Given that both hDHHC20 and zfDHHC15 have four TM helices, with the DHHC loop located between TM2 and TM3, this experiment rules out the possibility of any physiologically relevant topology of hDHHC20 or zfDHHC15 that positions the DHHC loop in the lumen of the Golgi. The four TM helices of hDHHC20 and zfDHHS15 adopt a tepee-like organization in the membrane, coming close together on the lumenal side and splaying apart on the cytoplasmic side, where substrate engagement and catalysis take place (Fig. 2). TM1 and TM4 are the most tilted with respect to the transmembrane normal. On the lumenal side, short loops connect TM1 and TM2, and TM3 and TM4. On the cytoplasmic side, the DHHC cysteine-rich domain connects TM2 and TM3. The N-terminal part of this domain consists of a helix-turn-helix that is connected to TM2 by a flexible linker. The C-terminal part of the domain has three stacked β hairpins arranged roughly in parallel with the bilayer and terminates in a short linker that is connected to TM3. The C-terminal domain of hDHHC20 is also at the cytoplasmic face and begins with a short helical segment (αʹ1) that wedges in between TM3 and the β hairpins in the DHHC domain. This domain is connected to a β hairpin that protrudes into the cytoplasm, but comes back to the membrane proximal region and ends in an amphipathic helix (αʹ2) that rests against TM3 and TM4. On the C-terminal side of the amphipathic helix, a hydrophobic loop with a short α-helical segment (αʹ3) inserts into the putative membrane bilayer region and makes extensive contacts with TM2 and TM3. Together αʹ2 and αʹ3 form a supporting belt on the cytoplasmic side of TM3 and TM4 (Fig. 2, A to C). (A) Cartoon representation of hDHHC20. The four TM helices are shown in green, the DHHC cysteine-rich domain in blue, and the C-terminal domain in brown. The two spheres represent the Zn2+ ions. (B) Same as (A) for zfDHHS15. An extra helix in the C-terminal domain is labeled in orange. (C) Diagram of the secondary structure elements of hDHHC20, showing helices as rectangles and β sheets as arrows. The two gray circles represent Zn2+ ions, and the dashed line indicates disordered C-terminal domain not observed in the crystal structure. Colors are the same as in (A). (D) Molecular surface representation (left) of hDHHC20 colored by electrostatic potential, showing a distinctive basic patch (black arrow) and a cross section (right) showing a cavity (cyan arrow) that exists above the active site. Electrostatic surface potential was generated using an Adaptive Poisson-Boltzmann Solver (APBS) server with default settings. Positively (blue) and negatively (red) charged surfaces are displayed at the contour levels of +5 and –5 kBT/e, respectively, where kB is the Boltzmann constant, T is temperature, and e is the charge on an electron. The cysteine-rich domain binds two Zn2+ ions, which is consistent with data from other DHHC members suggesting that this domain contains zinc finger motifs (29). The Zn2+ ions presumably play a structural role, as they are positioned between three parallel layers of β hairpins (Fig. 3A). Three cysteines and a histidine form the tetrahedral coordination environment of each Zn2+, resembling canonical zinc finger motifs (37). Neither of the Zn2+ ions coordinate the active site cysteine and thus do not directly participate in catalysis (38). However, mutations of the coordinating residues have been shown to drastically change the stability and/or catalytic activity of DHHC3 (33), supporting a role for the Zn2+ ions in positioning the catalytic cysteine optimally. Consequently, these residues are conserved among all members of the DHHC family (fig. S1). Additionally, a role for the zinc fingers in the binding of protein substrates can also not be ruled out (39). (A) Close-up of the Zn2+ binding region showing coordination of each Zn2+ ion by three cysteines and one histidine in a CCHC configuration. (B) Close-up of the DHHC enzyme active site showing the catalytic Cys156 pointing upward toward a hydrophobic groove. Also shown are the aspartic acid and the first histidine of the DHHC motif and the Thr241 of the TTXE motif. Hydrogen-bonding interactions are shown with dotted lines. Trp158, Phe171, and Phe174, which form the base of the acyl-binding cavity in Fig. 2D, are also shown in stick rendition. (C) Analysis of the enzymatic activity of selected mutants of the active site residues shown in (B). The coupled-enzyme assay was used, and Michaelis-Menten fits are shown. AHHC, DAHC, and F171A mutant curves all overlay with essentially no enzyme activity. Data are mean ± SEM of two independent measurements. Single-letter abbreviations for amino acid residues are as follows: A, Ala; C, Cys; D, Asp; E, Glu; F, Phe; G, Gly; H, His; I, Ile; K, Lys; L, Leu; M, Met; N, Asn; P, Pro; Q, Gln; R, Arg; S, Ser; T, Thr; V, Val; W, Trp; and Y, Tyr. The active site is situated at the membrane-aqueous interface in agreement with the observation that proteins are palmitoylated at membrane-proximal cysteines (20). The catalytic DHHC motif (the two histidines are His154 and His155) is located on the β hairpins (β5-β6) that coordinate the Zn2+ ions (Fig. 3B). The aspartic acid and the His154 are on the top face of the β hairpin, protruding in the direction of the membrane, and presumably form part of a catalytic triad (40) where the aspartic acid forms a hydrogen-bonding interaction with the histidine (Fig. 3B). The His155 side chain coordinates a Zn2+ ion on the other face of the β hairpin and presumably positions the catalytic cysteine for nucleophilic attack. In the zfDHHS15 structure, the side chain of the active site serine residue rotates to form a hydrogen-bonding interaction with a main-chain amide nitrogen (fig. S9). However, the putative position of the catalytic cysteine in the wild-type enzyme will likely bring it within hydrogen-bonding distance of His154. The structure suggests that the aspartic acid and the first histidine of the DHHC motif serve crucial roles in activating the nucleophilic cysteine. Mutating these residues individually to alanine generates mutants with no discernible catalytic activity (Fig. 3C). This organization of the active site is such that the catalytic cysteine of the DHHC motif protrudes in the direction of the membrane bilayer (Fig. 3B). Given that DHHC enzymes catalyze via a two-step reaction mechanism and that the acylated enzyme is an intermediate species with a long hydrophobic moiety attached to it, this suggests that the active site is preorganized to stabilize the acylated intermediate by inserting the acyl chain into the hydrophobic part of the bilayer. hDHHC20 has two positively charged patches on the cytoplasmic side (Fig. 2D, left); one of them arises from the binding of the two positively charged zinc ions. Another basic patch containing Arg126 and Lys135 showed electron density that we modeled as adenosine 5′-diphosphate with an additional phosphate at the 3′ end (5′-diphosphoadenosine 3′-phosphate) (fig. S10, A to C). This was based on consideration of components in the crystallization solution and other possible carryover contaminants such as adenosine 5′-triphosphate (ATP). Interestingly, this moiety is contained in the terminal part of CoA and thus could represent a byproduct of catalysis (fig. S10E). Because we did not add CoA during the protein purification, presumably this ligand was carried through the purification process. Notably, acyl-CoA has a long pantothenyl linker connecting thioester and the ribose ring. The extended form of this linker can easily span the length between the active site and the 5′-diphosphoadenosine 3′-phosphate binding site in the structure (fig. S10D). Based on these considerations, we suggest the 5′-diphosphoadenosine 3′-phosphate binding site to be the place where the terminal part of the coenzyme headgroup of palmitoyl-CoA binds. Although the C-terminal domain in DHHC enzymes is more variable compared to the cysteine-rich domain, it contains sequence motifs that are conserved to varying extents throughout the family. The most highly conserved among them is the TTXE (Thr-Thr-X-Glu) motif that resides in the short helical segment (αʹ1) immediately following TM4, where X is any amino acid residue. It makes intimate contact with the DHHC domain (Figs. 3B and 4A and fig. S1). The Thr240 of the conserved TTXE motif caps a main-chain amide nitrogen, and Thr241 forms a hydrogen bond with the aspartic acid side chain of the catalytic DHHC motif (Figs. 3B and 4A and fig. S9B). Consequently, an AAXE (Ala-Ala-X-Glu) mutant has drastically reduced enzymatic activity (Fig. 3C). (A) Interactions of the Thr240, Thr241, and Glu243 of the conserved TTXE motif with the active site and the rest of the protein. (B) Interactions of the amphipathic helix of the PaCCT domain with TM3 and TM4. Asn266 is one of the most conserved residues on the C-terminal domain of DHHC enzymes. (C) Penetration of the C-terminal short helical stretch αʹ3 together with the hydrophobic loop, shown in brown, into the bilayer, and their interactions with the cytoplasmic ends of TM2 and TM3. Relevant residues are shown in stick rendition. (D) Mutational analysis of selected residues on the C terminus involved in interactions shown in (A), (B), and (C). Data are mean ± SEM of two independent measurements. Montoro et al. (41) have identified a motif at the C terminus of DHHC enzymes that they have named the palmitoyltransferase conserved C-terminus (PaCCT) motif. This motif is conserved in the majority of DHHC enzymes, and, in the 16-residue motif, the third and the eleventh residues are the most conserved. The third residue is usually a phenylalanine or tyrosine and, in hDHHC20, this residue, Phe259, forms a local packing core at the C terminus and is engaged in a number of hydrophobic and π-stacking interactions. The other conserved residue in this motif, Asn266, is at the beginning of the amphipathic helix (αʹ2) and simultaneously forms a capping interaction with TM4 and a hydrogen-bonding interaction with the backbone of Leu261 and the sidechain of Ser260. These interactions engage all of the hydrogen-bonding capabilities of the amide side chain and presumably hold this part of the structure together. Mutating Asn266 to alanine severely compromises catalytic activity (Fig. 4, B and D, and table S3). One of the most prominent features of the C-terminal domain is the presence of an amphipathic helix (αʹ2) that makes contact with TM3 and TM4 and likely provides local stability. Sequence conservation indicates that the amphipathic helix is conserved across many members of the family, pointing to its importance for other DHHC enzymes as well. Deletion of a single phenylalanine residue in this region in DHHC21 has been genetically mapped to a depilated phenotype in mice that results in hair loss with thinner and shorter hairs (42). The homologous residue in hDHHC20 is Trp267. Deletion of Trp267 in hDHHC20 results in decreased yield of the recombinant protein and almost negligible catalytic activity (Fig. 4, B and D). The amphipathic helix is followed by a short helix (αʹ3) and a hydrophobic loop. The conformation of the loop is stabilized by highly conserved lysine and proline residues, and it inserts hydrophobic residues into the putative lipid bilayer and forms additional contacts with TM3 and TM2 (Fig. 4C). Mutation of the conserved Trp278 and Leu279 to a W278A/L279A double mutant reduces hDHHC20 activity by almost half (Fig. 4D and table S3). Starting at the active site residues and pointing inward into the bilayer, both hDHHC20 and zfDHHC15 have a hydrophobic cavity (Fig. 2D). In the hDHHC20 structure, we saw weak density in the cavity that was suggestive of a fatty acid, but was not strong enough to build a model. We thus utilized the widely used covalent inhibitor 2-bromopalmitate (2-BP) to purify an acylated intermediate mimic. This inhibitor covalently modifies DHHC enzymes at the active site cysteine through a nucleophilic displacement reaction (43, 44) (fig. S3B), resulting in alkylation of the cysteine through attachment at the α position of palmitic acid. Our initial biochemical and mass spectrometric data showed that, aside from the active site cysteine (Cys156), another cysteine (Cys263) was also consistently labeled with palmitate (fig. S11). Consequently, we used a C263A mutant (cysteine replaced with alanine at position 263) for preparation and crystallization of the covalently modified hDHHC20. We detected very low activity of the purified 2-BP–treated C263A mutant of hDHHC20, suggesting that the catalytic cysteine was mostly labeled in our preparation (fig. S3C). It crystallized under the same conditions and in the same hexagonal space group as hDHHC20 (table S1). The overall structures are very similar (Figs. 2A and 5A), and an omit map revealed clear density for the palmitate chain covalently attached to the active site cysteine, allowing us to build the complete acyl chain (Fig. 5A). In the zfDHHS15 structure, we saw continuous density in this cavity, which we modeled as a palmitic acid (fig. S12). (A) 2Fo-Fc electron density map at contour level 1.0 σ (magenta) and 0.7 σ (cyan) of the palmitate chain covalently attached to Cys156 in the hDHHC20–2-BP structure (see text). (B) Close-up view of the acyl chain binding groove in the 2-BP structure showing the residues lining the groove. (C). Residues in the groove that interact with the palmitate chain identified by using the small-probe contact dot surfaces (67). (D) Mutational analysis of the residues in the acyl binding groove shown in (C). Data are mean ± SEM of two independent measurements. (E) Acyl-CoA chain–length selectivity in wild-type and Y181A mutant of hDHHC20, as determined by the autoacylation assay. The x axis shows the carbon-chain lengths of different acyl-CoA donors, and the y axis shows normalized activity (initial velocity) of wild-type or mutant versions of hDHHC20. Each data set is individually normalized to 1 for the activity with regard to palmitoyl-CoA. (F) Acyl chain–length selectivity of S29F hDHHC20. Selectivity data are mean ± SEM of two independent measurements. The structure of the 2-BP–treated C263A hDHHC20 mutant (hereafter referred to as 2-BP structure) reveals crucial insights into the mechanism of acyl-chain binding and recognition in hDHHC20. The acyl chain inserts into the hydrophobic cavity seen both in the hDHHC20 and zfDHHC15 structures (Fig. 2D and fig. S9) and is contacted by several residues lining the cavity (Fig. 5, B and C, and fig. S12, B and C). The acyl chains have very similar conformation in the cavity, and we will focus our description on the hDHHC20 2-BP structure. Notably, all four TM helices contribute to interactions with the acyl chain, albeit to different extents. The cavity is constricted near the acyl headgroup by Trp158 and Phe171. These are two of the most highly conserved residues across the DHHC family members (fig. S1). Mutation of Phe171 to alanine generates a catalytically inactive mutant, whereas mutation of Trp158 to alanine causes a considerable loss in enzymatic activity (Fig. 3C and table S3). Further into the membrane, two other residues contact the acyl chain, Phe174 and Leu227, conserved as hydrophobic residues in most DHHC members. Mutation of the leucine to a more bulky and rigid tryptophan severely compromises enzymatic activity (Fig. 3C and table S3). Toward the top of the cavity where it tapers to a narrow end, residues from all four TM helices contact the acyl chain. Some of these residues that line the acyl binding cavity are highly conserved across all DHHC members, e.g., Trp28 in hDHHC20. Intriguingly, there are other residues that show only subgroup-specific conservation (fig. S1). As a case study, Ile22 is conserved among a subset of DHHC members and faces the acyl binding cavity in the structure, but does not make a close contact (Fig. 5B). The I22W mutant (isoleucine replaced with tryptophan at position 22) has drastically lower activity, presumably owing to the more bulky and rigid aromatic side chain of tryptophan, which makes a disfavorable contact with the acyl chain (Fig. 5D). It has been shown that different DHHC members show different degrees of selectivity for acyl-CoA donors with different chain lengths (28, 30). Toward the tapering end of the acyl binding cavity in the 2-BP structure, Tyr181 forms a hydrogen-bonding interaction with Ser29, effectively closing off the cavity (Fig. 5B). Whereas wild-type hDHHC20 shows a preference for acyl-CoAs with a palmitoyl (16-carbon) chain, mutating Tyr181 to a less bulky alanine results in marked increase in preference for stearoyl (C18)–CoA (Fig. 5E). Mutating Ser29 to a bulkier phenylalanine increases the preference of hDHHC20 for short-chain acyl-CoA (Fig. 5F). The S29F (serine replaced with phenylalanine at position 29) mutation not only breaks the hydrogen-bonding interaction but also replaces the Tyr-Ser pair in hDHHC20 with a considerably bulkier Tyr-Phe pair in the cavity. These results also lend insights into a recent report that investigated the acyl chain–length selectivities of two closely related DHHC enzymes, DHHC3 and DHHC7 (30). The homologous pair of Tyr181-Ser29 in hDHHC20 is Ile182-Phe53 in DHHC3 and Ser185-Leu56 in DHHC7. DHHC3 has a higher selectivity for palmitoyl (C16) over stearoyl (C18) compared to DHHC7. Presumably this is due to the placement of two large residues (isoleucine and phenylalanine) toward the end of the cavity, which impedes longer–chain length acyl-CoA. In comparison, the less sterically demanding Ser-Leu pair leads to a more relaxed acyl chain–length preference in DHHC7. Consistent with this hypothesis, the I182S (isoleucine replaced with serine at position 182) mutant of DHHC3 has higher preference for stearoyl (C18)–CoA, mirroring that observed in DHHC7 (30). Although protein palmitoylation was discovered almost 40 years ago (5), there has been little understanding about the structural chemistry of enzyme-catalyzed protein palmitoylation. All of the DHHC enzymes that have been biochemically characterized to date utilize a two-step reaction mechanism (27, 28), with the first step being autoacylation of the active site cysteine. We show that the aspartic acid and His154 of the DHHC motif form a hydrogen-bonded pair that can accept a proton from the cysteine within the DHHC motif during catalysis, enabling nucleophilic attack on the carbonyl thioester of palmitoyl-CoA to generate the acyl-enzyme intermediate (Fig. 6A). The structures reported here do not give indication as to how the palmitoyl-CoA enters the binding cavity. However, palmitoyl-CoA partitions into the lipid bilayer and likely distorts the local structure of the membrane (45, 46). Given that interfaces of membrane proteins with surrounding lipids are also sites of distortion in the lipid bilayer and, thus, possible sites of accumulation of palmitoyl-CoA, we speculate that palmitoyl-CoA inserts into the cavity formed by DHHC enzymes from within the membrane. (A) Proposed reaction mechanism of DHHC enzymes that follow a two-step mechanism. (B) Molecular surface rendition of the 2-BP–treated enzyme with the acyl chain shown in yellow and red spheres. The putative direction of substrate approach is shown with a cyan arrow. In the second step of the reaction, the acyl-enzyme intermediate transfers the palmitoyl group to the substrate cysteine. The structure of the 2-BP–treated DHHC20 suggests that the carbonyl oxygen of the acyl-enzyme thioester is positioned close to His154. We speculate that the protonated His154 activates the acyl-enzyme thioester for the subsequent palmitoyl transfer to the substrate by providing a proton to the carbonyl oxygen. The cysteine residue in each substrate that accepts the palmitoyl group lies in a different sequence context and, thus, different chemical and structural microenvironments. Consequently, activating the acyl group of the acyl-enzyme intermediate with an adjacent proton donor would be an effective strategy for catalysis of the second step. Residue His154, which accepts a proton in the first step, is optimally positioned for this role. The structure also shows that the organization of the active site is such that all but one side of the acyl-enzyme thioester is shielded by hydrophobic residues. This only leaves the front side of the acylated DHHC for approach of the cysteine residue on the substrate that reacts with the thioester in the second step (Fig. 6B). However, engagement with the substrate may incur conformational changes and open up alternative directions of approach that are not obvious in the current set of structures. Protein S-acylation has been shown to be heterogeneous. Analyses of S-acylated proteins from native sources have revealed that proteins can be modified not only with palmitic acid but also with other fatty acids, such as stearic acid and oleic acid (10, 11, 47). In the case of influenza hemagglutinin, specific cysteines have been shown to be modified by palmitoylation and stearoylation, pointing to the specific roles that distinct acylation states play in the function of cellular proteins (48). Recently, it has been shown that stearoylation of transferrin receptor 1 by DHHC6, specifically, modulates mitochondrial morphology (49). In vitro experiments have shown that for the two enzymes DHHC2 and DHHC3, DHHC2 can utilize a broad range of fatty acids with diverse structures such as palmitate, arachidonate, and palmitoleate, whereas DHHC3 has reduced activity for fatty acids longer than palmitic acid (28). It was also shown that the selectivity at the autoacylation state reflects the selectivity of fatty acyl labeling of the protein substrate (28). The structures of unacylated zfDHHS15 and hDHHC20 and of 2-BP hDHHC20 reveal that the fatty acyl chain fits into a cavity in the bilayer formed by the transmembrane helices of the enzyme. The residues lining this cavity form numerous contacts with the fatty acyl chain. However, although there are highly conserved residues, i.e., Trp158 and Phe171, lining this cavity that form important contacts with the acyl chain, there is considerable sequence diversity in the other residues that line this cavity between different subgroups of DHHC enzymes (fig. S1). These residues vary not only in their size, but also in their polarity, thus changing the chemical properties of the cavity in a complex manner. These can have a notable effect on the fatty acyl chain–length preference of different DHHC enzymes, and, as structures of more DHHC enzymes become available, presumably they will reveal further insights into their respective fatty acid chain–length selectivity. DHHC20 overexpression was previously shown to cause cellular transformation (50). More recently, hDHHC20 has been proposed as a target for developing therapeutics against cancers that are resistant to EGFR-targeted therapy (22). The structures presented here are plausible starting points for developing a structure-guided program to hDHHC20 inhibitors that could lead to such a therapeutic intervention. It is worthwhile to note in this context that the entire field of protein palmitoylation suffers from the lack of small molecule probes and inhibitors for the DHHC family of enzymes, let alone the lack of specific modulators of individual DHHC members (51). Members of the DHHC family all use palmitoyl (acyl)–CoA and the same enzymatic chemistry to transfer an acyl group to substrate proteins. From this perspective, they are very similar to members of the family of protein kinases that use ATP to transfer a phosphate group to substrate proteins. Thus, using strategies like bump-hole pairs, where structural knowledge is used to design engineered versions of DHHC enzymes that can use orthogonal acyl-CoAs, could provide transformative insights into the biology of protein palmitoylation and help unravel the complex network of DHHC-substrate interactions, akin to the field of protein kinases and histone-modification enzymes (52–54). We demonstrate the feasibility of such an approach by designing mutants that can shift the acyl chain–length selectivity profile of the mutants toward the shorter or longer side of palmitoyl (C16)–CoA. The codon optimized zebrafish DHHC15 (zfDHHC15) sequence was cloned into a modified version of pPICZ-C vector with an His10 tag followed by a GFP coding sequence and finally, by PreScission cleavage site at the N terminus of the zfDHHC15 encoding DNA sequence. Further, site directed mutagenesis was performed to mutate Cys153 to serine to get the zfDHHS15 expression construct. The optimized vector harboring zfDHHC15 or zfDHHS15 gene was transformed into Pichia pastoris HIS + Cells SDM1163 cells. Pichia were transformed using standard methods and the transformants were selected on YPDS plates containing 400-800 μg/mL zeocin. The human DHHC20 (hDHHC20) expression construct was made similarly with N-terminal His10 tag, GFP and PreScission cleavage sequences without any modification/truncation of the hDHHC20 sequence and was transformed into Pichia following the same protocol. A vector containing the sequence for human Snap25b was obtained from Addgene (Plasmid #53235) and the gene was cloned into pET28-Prx by digestion/ligation using NdeI/BamHI. The original TEV cleavage site was mutated to a PreScission cleavage site. Large scale culture of Pichia, induction of protein expression and cell lysis were carried out as described previously (55). Briefly, 100-200 mL BMGY (0.1 M Potassium phosphate, pH 6.0, 3.4 g/L yeast nitrogen base, 1% glycerol, 0.4 μg/mL biotin) with 500 μg/mL Zeocin cultures of Pichia were grown overnight at 30°C with vigorous shaking. 1/10th volume of these starter cultures, which usually reached cell densities of OD600 ~20, were then used to inoculate 1.5-2 liter cultures in the same media but without the Zeocin. In 24-36 hours, the cells were pelleted by centrifugation at 1500 g, 4°C. for 10 min and thoroughly washed with BMMY (0.1 M Potassium phosphate, pH 6.0, 3.4 g/L yeast nitrogen base, 1% methanol, 0.4 μg/mL biotin). Cells were finally re-suspended in 1.5-2 liters of BMMY media and protein expression induced at 22-24°C for 24-36 hours. Cells were harvested by centrifugation at 6000 g, 4°C. for 20 min. The cell pellets were scooped by a spatula and frozen by immersing into liquid N2 followed by storage at –80°C. Cells were lysed by cryo-milling using Retsch MM400 millers with liquid N2 for cooling. Lysed and frozen yeast powder was dissolved in the lysis buffer (1 g cells per 3 mL of lysis buffer) containing 50 mM Tris-HCl pH-7.5, 150 mM NaCl, 1 mM TCEP, 0.1 mg/mL deoxyribonuclease I, 0.1 mg/mL pepstatin, 1 μg/mL leupeptin, 1 μg/mL aprotinin, 1 mM benzamidine and 0.1 mg/mL soy trypsin inhibitor. The pH of the cell lysate was adjusted to 7.8 followed by addition of the 0.14 g of DDM (n-dodecyl-β-d-maltopyranoside) per g of cells. The proteins were extracted from the cells by stirring at 4°C. for 3 hours. The pH of the cell lysate was adjusted to 7.5 and centrifuged at 38,000 g, 4°C. for 30 min. The supernatant was allowed to bind with 2.5 mL of Talon (Clontech) resin pre-equilibrated with the equilibration buffer (50 mM Tris-Cl pH 7.5, 150 mM NaCl, 1 mM TCEP and 1 mM DDM) for 3 hours at 4°C. The protein bound Talon resin was collected in a Bio-Rad Econo column and washed with 10 bed volumes of wash buffer 1 (20 mM Tris-Cl pH 7.5, 150 mM NaCl, 1 mM TCEP and 2 mM of Thio-DM (n-Decyl- β-d-Thiomaltopyranoside) followed by additional wash with 10 bed volumes of wash buffer 2 (wash buffer 1+ 20 mM imidazole). An additional wash was performed with 10 bed volumes of wash buffer 1 in order to completely wash away any traces of imidazole. Talon resin was resuspended in 10 mL of wash buffer 1. His10 tagged GFP was removed from the N terminus of zfDHHS15 by incubation of the resin slurry with PreScission protease overnight at 4°C. Flow-through containing cleaved protein was concentrated with 50 kDa molecular weight cut-off (MWCO) 15 mL concentrators (Millipore). The concentrated protein sample was further purified by size exclusion chromatography on Superdex 200 column in buffer containing 20 mM Tris-Cl pH 7.5, 150 mM NaCl, 1 mM TCEP and 2 mM Thio-DM. The peak fractions from the size exclusion chromatogram were collected and concentrated with 50 kDa MWCO concentrator up to 20 mg/mL and used for crystallization. 20-24 g of milled cell powder was suspended in ~65 mL of Tris-buffered saline and stirred using a magnetic stirrer for 20-30 min to make a homogeneous slurry. The final buffer composition was 40 mM Tris.HCl, 270mM NaCl, 5 mM βME. Protease inhibitors (Benzamidine HCl, PMSF, AEBSF, Aprotinin, Pepstatin, Leupeptin) and DNase were added and the pH of the cell slurry adjusted to ~7.5. Solid DDM was added to give a final concentration of 2% (40mM) and the protein extracted from the membranes by vigorous stirring at 4°C for 3 hours. Cell debris was pelleted by centrifugation at 38,000 g for 30 min at 4°C. The pH of the supernatant was adjusted to 7.5 and then incubated with 2 mL of Talon resin for 2-3 hours at 4°C. Protein-bound resin was washed with 30 mL of 50 mM Hepes, pH 7.5, 250 mM NaCl, 5 mM βME, 2 mM TCEP, 1 mM DDM buffer containing 5 mM imidazole. The resin was further washed with same volume of buffer containing 25 mM imidazole prior to elution with ~5-7 mL of buffer with 200 mM imidazole. All affinity chromatography was done by gravity flow at ambient temperature with buffers chilled on ice. The bright green eluate was then concentrated in a 100 kDa molecular weight cut-off centrifugal concentrator to ~0.3 mL. 1/10 volume PreScission protease (~1 mg/mL) was then added and rotated overnight at 4°C. The PreScission protease cleaved protein was injected into a Superdex 200 Increase size exclusion column at 4°C. to separate hDHHC20 from the GFP tag and the PreScission protease. The size exclusion buffer was 50 mM Hepes pH 7.5, 150 mM NaCl, 2 mM TCEP, 0.5 mM DDM. The fractions containing the protein were pooled and concentrated using a 50 kDa MWCO 0.5 mL Amicon centrifugal concentrator. Prior to any assays, protein concentration was determined using the 660 nm protein assay kit (ThermoFisher). In cases where the GFP tag was not removed, the overnight PreScission cleavage step was omitted and the concentrated protein applied to the Superdex 200 increase column (GE Healthcare). hDHHC20 for crystallization was prepared as above to the step of overnight PreScission digest. The next morning, instead of applying the GFP-cleaved protein to the gel filtration column, it was diluted to ~12 mL in 50 mM Hepes, pH 7.5, 250 mM NaCl, 2 mM TCEP, 0.5 mM DDM buffer lacking any imidazole. The GFP tag was then removed by binding it to ~1 mL Talon resin and the flowthrough containing the hDHHC20 protein was concentrated in 50 kDa MWCO 4 mL concentrator to ~0.5 mL. This protein was again diluted to ~12 mL and re-concentrated to ~1 mL to remove any imidazole. The protein was then reductively methylated using formaldehyde and dimethylamineborane (DMAB) complex following standard protocols (56). The next morning, the methylation reaction was quenched with 100 μl of 1 M Tris HCl, pH 8.0 and 5 μl of 1 M dithiothreitol (DTT) for 1 hour at 4°C. The protein was concentrated to ~0.25 mL and applied to Superdex 200 Increase size-exclusion column as previously mentioned. Peak fractions were collected and concentrated as before. All wash buffers for the crystallization sample of DHHC20 contained 0.1 mg/mL POPC:POPG:POPA (3:1:1) lipids and 1 mM DDM. This was lowered in the size exclusion chromatography buffer to 0.05 mg/mL lipids and 0.5 mM DDM. To prepare an irreversibly inhibited protein with a covalently attached lipid chain, 2-bromopalmitate (2-BP) dissolved in methanol was added to the Pichia cells at a final concentration of 300 μM during induction. However, we decided to add the DHHC inhibitor only after the cells had been induced for at least 12 hours as 2-BP is toxic to cells and we wanted to ensure that it did not affect protein expression initially. Five hours prior to harvest, we again added 2-BP to a final concentration of ~100 μM. We first attempted this with the wild-type hDHHC20 and checked the covalent modification with mass spectrometry. Unexpectedly we found that apart from the active site cysteine, 2-bromopalmitate also modifies Cys263. Therefore, to reduce any possible heterogeneity during crystallization, we used a C263A mutant for the 2-BP modified DHHC20. hDHHC20 C263A protein was prepared as for wild-type hDHHC20. Enzymatic activity of the 2-BP treated sample was checked and found to be essentially nonexistent as expected. GobX protein expression vectors were a gift from M. Machner (NIH). BL21(DE3) Gold cells were transformed and plated on LB-agar with Ampicillin. Next morning the entire plate was used to start two 1 liter cultures in LB media. Bacteria were grown to an OD600 of ~0.6 at 37°C. with vigorous shaking. Temperature was reduced to 30°C. and protein expression was induced with a final IPTG concentration of 1mM. Cells were harvested ~5 hours later. Protease inhibitors (PMST, AEBSF) were added to the cells prior to flash freezing in liquid N2. Next morning the cells were thawed in 40 mM Tris-HCl, pH 7.2, 270 mM NaCl, 10 mM βME, 20% glycerol, DNAse, and more protease inhibitors. Powder lysozyme was added to facilitate cell lysis. Once the cells had thawed to slurry, the cell suspension was diluted such that it now contained 10% glycerol. DDM was then added to 1% final concentration. Cells were broken by sonication on ice. During the sonication process, 1/100 volume PMSF was added after every 1 min. The cell debris were removed by centrifugation at 38,000 g for 30 min at 4°C. The supernatant was incubated with 2 mL of HisPur Ni-NTA resin for 1-2 hours at 4°C rotator. Protein bound resin was packed on a Bio-Rad Econo column and washed with ~30 mL Tris-buffered saline containing 10% glycerol and 10 mM imidazole. The resin was further washed with same buffer containing 50 mM imidazole before finally eluting it with ~8 mL 300 mM imidazole buffer. The eluate was concentrated in 10 kDa MWCO concentrator to ~0.5 mL and diluted 100-fold to 50 mL in 50 mM TrisHCl, pH 8.0, 10% glycerol, 1 mM DDM buffer. The diluted protein was applied to a 1 mL HiTrap MonoQ column using peristaltic pump. GobX protein was eluted off the MonoQ with a two-gradient program between 50 mM TrisHCl, pH 8.0, 10% glycerol, 10 mM NaCl and same buffer with 1 M NaCl on an AKTA explorer at 4°C. DDM is omitted from the MonoQ column elution buffers. Two major peaks are eluted – the first is a cleaved version of the GobX and the second is the full-length GobX. The second peak fractions were pooled, concentrated in a 10 kDa MWCO Amicon spin concentrator to ~5-7 mg/mL and flash frozen in liquid N2 until later use in palmitoylation assays. E. coli BL21(DE3)-Gold was transformed with Snap25b in pET28-Prx vector and plated onto LB-Kanamycin plates. A single colony was used to inoculate 5mL of LB (with 50 μg/mL of Kanamycin) and cells were grown at 37°C for 20 hours. 2 mL of this starter culture was used to inoculate 1 L of LB (with 50 μg/mL of Kanamycin) and cells were grown until OD600 was around 1.0. Cells were cooled to 30°C. and induced with 0.5 mM of IPTG. After 5 hours, cells were harvested by centrifugation at 8000 g for 10 min at 4°C. Cell pellets were flash frozen in liquid nitrogen and stored at –80°C. For a typical purification, 10 g of frozen pellet were dissolved by stirring at 4°C (30-40 min) in 100mL of lysis buffer containing: 50 mM Tris-HCl pH 8.0, 250 mM NaCl, 5% glycerol, 5 mM βME, protease inhibitor cocktail (0.1 mg/mL deoxyribonuclease I, 0.1 mg/mL pepstatin, 1 μg/mL leupeptin, 1 μg/mL aprotinin, 1 mM benzamidine and 0.1 mg/mL soy trypsin inhibitor and 1 mM PMSF) and 10 mg lysozyme. Resuspended cells were disrupted by sonication, stirred for 20 min at 4°C and subsequently centrifuged at 38,000 g for 30 min at 4°C. Pellets were discarded and supernatant was batch-bound to 5 mL of Ni-NTA resin (previously equilibrated with binding buffer containing 50 mM Tris-HCl pH 8.0, 250 mM NaCl, 5% glycerol, 5 mM β-mercaptoethanol) by rotation at 4°C. for 1 hour. Protein-bound resin was washed successively with 10 bed volumes of binding buffer, 10 bed volumes of binding buffer with 10 mM imidazole and finally, 10 bed volumes of binding buffer with 30 mM imidazole. Protein was finally eluted with binding buffer containing 250 mM imidazole. Eluted protein was incubated at 4°C with rotation for 20 hours in the presence of 1 mg/mL of PreScission protease to remove the His6 tag. Completeness of cleavage was assessed by gel shift using 12% acrylamide SDS-PAGE after Coomassie staining. Completed cleavage reaction was dialyzed against 4 liters of binding buffer for 15 hours at 4°C to remove imidazole. The dialyzed protein solution was re-applied to 5 mL of Ni-NTA resin equilibrated with binding buffer and incubated with rotation at 4°C for 1 hour. Resin was loaded onto a column and flow-through was collected. The concentration of purified protein was determined using the 660nm Assay Kit (Thermo Fisher). hDHHC20 protein at ~15-20 mg/mL was incorporated into monoolein using established protocols (57) and crystallization trials were set up on Laminex 96-well plastic sandwich plates (Molecular Dimensions) using a Mosquito crystallization robot (TTP Labtech). An initial hit was obtained with 0.1 M Hepes, pH 7.0 and 30% PEG 300 at 20°C. Optimization of crystals was carried out using varying salts, buffers and pH. Of the salts, only sodium malonate and potassium di-hydrogen phosphate at 50 mM gave crystals that diffracted consistently. Further improvements in size of crystals were achieved through screening diol additives with 1-5% of 2,5-Hexanediol being the best. Better diffraction was observed when fresh DTT was included as an additive at concentrations of 25-50 mM. Addition of lipids during the protein preparation improved the crystallizability of DHHC20. While we had initially observed rectangular crystals (indexed as P21), inclusion of lipids during purification led to us to obtain hexagonal crystals (indexed as P63) as well. Further improvements in diffraction were also achieved by methylating the protein. The final optimized crystallization screen was a pH and PEG 300 grid with MES (pH 5.5-6.5), MOPS (pH 6.5-7.5), and HEPES (pH 6.5-7.5) buffers and PEG 300 from ~25-35%. We also expanded the salt from potassium di-hydrogen phosphate to include the sodium salt as well, both at 50 mM final concentration. As diffraction from crystals obtained in the sandwich plates hardly extended to better than 4 Å and the harvesting itself was challenging, we decided to switch to a microbatch format. In this format, 100 nl of the LCP bolus was set on 96-well Microbatch plates (Hampton Research) followed by 3.24 μl crystallization solution and 0.36 μl of a 25% 2,5-Hexanediol and 0.5 DTT (freshly prepared) mixture. Plates were incubated at 20°C. The crystals generally took longer to appear than sandwich plates (a week versus a few days) and also grew larger and thicker. Crystals were directly fished out of the wells using Mitegen 100 micron loops and frozen in liquid N2. The initial hexagonal crystals were obtained in 100 mM HEPES, pH 7.3, 50 mM NaH2PO4, 25 mM DTT, 1.25% 2,5-hexanediol after approximately one week. Data sets were collected from hexagonal crystals at 9.665 keV at 10% transmission with 0.25° rotation and 0.2 s exposure. A total of 720 frames spanning 180° were collected. The best diffracting hexagonal crystals were obtained in 50 mM MES, pH 5.8, 50 mM KH2PO4, 27.7% PEG 300, 25 mM DTT, 1.25% 2,5-Hexanediol and grew to optimal size in 3.5 months. A native data set of 360 frames was collected at Se-edge of 12.66 keV with 10% transmission, 0.25° rotation and 0.5 s exposure. These data sets were collected at the NE-CAT 24ID-C beamline at the Advanced Photon Source (APS) at Argonne National Laboratory in Chicago, IL. The best P21 crystal was obtained in 50 mM MES, pH 6.5, 50 mM NaH2PO4, 30.3% PEG 300, 50 mM DTT, 2.5% 2,5-hexanediol and grew to optimal size in 3 months. Complete data set was collected at 9.665 keV with 10% transmission, 0.3° rotation, and 0.2 s exposure. A total of 1500 frames were collected. The data set for the 2-BP modified hDHHC20 crystals was obtained from a single crystal obtained after 1.5 months in 100 mM HEPES, pH 7.03, 50 mM KH2PO4, 31% PEG 300, 50 mM DTT, and 2.5% 2,5-hexanediol. Data set of 900 frames was obtained with 20% transmission, 0.2° rotation, and 0.2 s exposure at a 12.66 keV beam. These data sets were collected at the GM/CA-CAT 23ID-D and 23ID-B beamlines respectively at the APS at Argonne National Laboratory in Chicago, IL. Diffraction data from three different crystals of the P63 form were merged together to improve redundancy for the phasing. The data sets were processed using XDS and AIMLESS in the CCP4 suite (58). Initial phases were obtained using AutoSol in the PHENIX Suite (59). Four predicted zinc heavy atom sites were identified and provided a phasing power to generate a map clearly showing molecular boundaries and secondary structure elements. An initial model was built using AutoBuild and manually modified using COOT (60). The resulting structure was used as a search model for molecular replacement against the native data set. This structure was also used as a starting molecular replacement model for solution of the P21 form and the 2-BP treated hDHHC20. Restraints of the 2-BP and CoA head group were generated using the eLBOW module within the PHENIX suite and edited manually. Iterative model building and refinement was carried out using COOT and PHENIX. The coordinates and structure factors of the human hDHHC20 in P63 and P21 space groups and 2-BP modified structure have been deposited to the Protein Data Bank (PDB) with the accession code of 6BMN, 6BMM and 6BML, respectively. ZfDHHS15 protein crystallization was performed using the hanging drop vapor diffusion method. Initial crystal hits were obtained with 0.1 M KCl, 20% PEG 400 and 100 mM HEPES, pH 6.5 at 4°C. These crystals were further optimized to obtain better diffraction quality crystals. The best diffracting crystals of zfDHHS15 grew from 0.1 M KCl, 20% PEG 400, 100 mM HEPES pH 6.5 and 30% ethylene glycol at 4°C. The optimum size of the better diffracting crystals was obtained in 7 days. Crystals were exposed to the cryo-solution (30% ethylene glycol with all other remaining same as reservoir) by exchanging reservoir solution with cryo-solution. Ethylene glycol percentage was gradually increased from 20% to 30% in increments of 5%. Each step was incubated at 4°C for 6-7 hrs. Crystals were harvested using Hampton nylon loops and immediately flash-frozen in liquid nitrogen. Three different types of data were collected in order to solve the structure. Two different single-wavelength anomalous dispersion data sets at 3.31-Å resolution from single crystals were collected at the Zn absorption edge wavelength, 1.2820 Å, on beamline 24ID-C (NE-CAT) at the Advanced Photon Source (APS), Argonne National Laboratory. These data sets were merged together, integrated with XDS and scaled with XSCALE. Additionally, S-SAD data were collected at 1.71 Å from 10 different single crystals at 23ID-B (GM-CA) at APS. These data sets were merged together and integrated with XDS and scaled with XSCALE. A third native data set of 2.54-Å resolution was acquired at x-ray wavelength 0.997 Å using single crystals on beamline 22ID (SER-CAT) at the APS. The native data set was processed with HKL 2000. The structure was solved using the Zn-SAD data in P42212 space group using the data collected at 24ID-C beamline, integrated using XDS. Data from two crystals were used. They were scaled using XSCALE. Four Zn atoms were located using hkl2map and subsequently refined using experimental phasing methods in Phaser. An electron density map was generated after NCS averaging and density modification (using Parrot, part of the CCP4 suite of crystallographic software) and an overall model was built into it. At this stage, a highly redundant S-SAD data was collected at 23ID-GM-CA (wavelength 1.71 Å, data redundancy ~100) using 10 crystals. Using the model phases, sulfur atoms were located using anomalous difference Fourier methods and the sulfur sites were refined using Phaser. A total of 48 sulfur sites were found (24 per monomer, 9 Met and 15 Cys). This density modified map was used for iterative model building and refinement in PHENIX. A partial model of the zfDHHS15 was obtained with Rwork of 32.2 and Rfree of 33.7%. This partial model was used for molecular replacement with native data set of 2.45 Å resolution with Phaser module in PHENIX. Non-crystallographic symmetry was used during refinement of the model. The native data set was further refined using PHENIX as well as Refmac in ccp4 followed by manual examination and rebuilding of the refined coordinates in coot using both 2mFo\\u2009−\\u2009DFc and mFo\\u2009−\\u2009DFc maps, as well as omit maps generated through PHENIX. The restraint parameters for palmitate, dodecylmaltoside and POPC were generated through ELBOW program of PHENIX was used during refinement. The final data collection and refinement statistics are shown in Table S2. The final model had the following Ramachandran statistics, 97.8% in favored region, 2.17% in allowed region, 0.2% of rotamer outliers and 0% of Ramachandran outliers. zfDHHC15 autoacylation activity was measured using a coupled-enzyme assay (31) in 384-well low-volume plates (Thermo Fisher). Plates were read in a Tecan M1000Pro fluorimeter with 340 nm excitation and 465 nm emission. The final reaction volume (20 μL) contained 20 nM zfDHHC15/zfDHHS15, 0.25 mM oxidized Nicotinamide adenine dinucleotide (NAD+), 0.2 mM thiamine pyrophosphate (TPP), 2 mM 2-oxoglutarate, 1 mM EDTA, 1 mM DTT, 2-oxoglutarate dehydrogenase, 0.6 mM DDM detergent in 50 mM sodium phosphate buffer at pH 6.8. The reaction was initiated by the addition of palmitoyl-CoA and monitored for 30 min at 30°C. The linear part of the progress curve was used to determine initial rates. zfDHHC15 concentration was determined using the 660 nm assay (Thermo Fisher). Km and Vmax were estimated using non-linear least square fitting to the Michaelis-Menten model (Graphpad Prism 6). Kcat was determined by dividing the estimated Vmax by the total enzyme concentration. Each Km and kcat value is the average and standard error of three independent measurements. For in vitro Snap25b acylation assay, affinity purified GFP-zfDHHC15 (50 nM) and GFP-zfDHHS15 (50 nM) were incubated with affinity purified Snap25b (500 nM) in reaction buffer (50 mM HEPES, pH 7.0, 200 mM NaCl, 0.2 μM DTT, 1 mM EDTA, 0.3 mM DDM) for 10 min at room temperature (24°C). Reaction was started by addition of {N-[(7-nitro-2-1,3-benzoxadiazol-4-yl)-methyl]amino}palmitoyl-CoA (NBD PalCoA) to a final concentration of 1 μM. NBD PalCoA is a fluorescent derivative of palmitoyl-CoA. Samples were taken at different time points and reaction quenched with non-reducing SDS sample buffer. Samples were separated in 12% SDS-PAGE gels and fluorescence detected using a Chemidoc with Blue Epi illumination and a 530/25 filter. hDHHC20 kinetic assays were done using an established fluorescence based coupled-enzyme assay (32) in 96-well format at 30°C. Plates were read in a Tecan M1000Pro plate reader with 340 nm excitation and 465 nm emission at 30 s intervals. All assays were done in 25 mM MES, pH 6.5, 50 mM NaCl, 1 mM DTT, 1 mM EDTA, 0.3 mM DDM. Final NAD+, TPP, and 2-oxoglutarate concentrations were 0.25 mM, 0.2 mM, and 2 mM respectively. Although the original assay used αKDH purchased from Sigma, we were dissatisfied with the activity and availability of the enzyme. Therefore, we prepared the αKDH in-house from beef heart using published protocol (61) αKDH quantity required for the assay was determined empirically by titration until the slope of the reaction progression curve did not change further. Palmitoyl-CoA and other acyl-CoA were usually kept below 20 μM as at higher concentrations we observed reduced enzyme activity. DHHC20 enzyme concentration in the assay to determine kinetic parameters was usually 2 nM to 5 nM. At higher DHHC20 concentrations it was difficult to get enough time points for low palmitoyl-CoA concentrations to determine the initial velocity. Km and kcat were determined using Michaelis-Menten equation using non-linear least squares fit in OriginLab. Curve-fitting was weighted to the error and was iteratively done to minimize reduced chi-square. Adjusted r-square values for most fits are >0.9 except for the inactive mutants where the data are not described by the Michaelis-Menten kinetics. For assays where initial slopes at low palmitoyl-CoA concentrations were not necessary or a stronger signal was more important, DHHC20 enzyme concentrations were increased to 10-100 nM. In such cases the acyl-CoA concentrations were usually fixed at 5 or 10 μM. GobX protein palmitoylation assays were carried out in 96-well plates at ambient temperature (~24°C). Reaction condition was 50 mM HEPES, pH 7.0, 200 mM NaCl, 0.2 μM DTT, 1 mM EDTA, 0.3 mM DDM. eGPF-DHHC20 and GobX concentrations were 0.2 and ~30 μM respectively (we used the eGFP-tagged DHHC20 because the DHHC20 and GobX protein overlap in the gel). After mixing the enzyme and substrate, trans-palmitoylation reaction was initiated by the addition of the fluorescent NBD–palmitoyl-CoA to a final concentration of 1 μM. 10 μl of reaction mixture was pipetted out at different time points and the reaction quenched by adding it to 2x non-reducing SDS-sample buffer. Quenched reactions were run on a SDS-PAGE and the NBD fluorescence imaged on a gel imager (Bio-Rad). Protein signal was obtained from Coomassie staining or the use of a stain-free gel (Bio-Rad). The Venus-hDHHC20 construct was inserted in the multiple cloning site of pEG-Bacmam vector (62) under the control of the CMV promoter. All mutagenesis was carried out using the Quikchange strategy. Venus-hDHHC20 wild-type and mutants were expressed in HEK293T cells grown in DMEM with 2 mM glutamine, 2% FBS, and 100 units/mL penicillin/streptomycin. 10 μg of DNA was complexed with 30 μg of polyethyleneimine (PEI) and used to transfect 100 mm plates of ~80% confluent HEK293T cells (63). Cells were harvested after 48 hours and washed with phosphate-buffered saline. The cell pellets were then stored at –20°C. for later use. In parallel 1 μg of the same DNA was also used to transfect HEK293T cells in 12-well plates. FSEC was used to assess the expression level and the gel filtration profile of the DHHC20 mutants. Following satisfactory assessment of FSEC profile, frozen HEK293T pellets were thawed on ice and re-suspended in 500 μl of 40 mM TrisHCl, pH 7.2, 270 mM NaCl, 5 mM 2-ME, 5 mM MgCl2 containing Dnase and protease inhibitors. 100 μl of a 0.3 M stock DDM was added to the cell suspension and rotated for 1-2 hours at 4°C. Cell debris was removed by centrifugation at 21,000 g for 10 min at 4°C. The supernatant was then applied to ~30-40 μl of TALON resin in a microfuge tube and incubated on a rotator for ~1 hour at 4°C. Resin was first washed with 800 μl of same buffer containing 5 mM imidazole and 1 mM DDM but no MgCl2. Following a second wash with 500 μl of buffer containing 25 mM imidazole, protein was eluted by ~40 μl of 20 mM TrisHCl, pH 7.2, 135 mM NaCl, 2 mM TCEP, 5 mM 2-ME buffer containing 300 mM imidazole. Protein concentration was estimated by running a stain-free SDS-PAGE (Bio-Rad) gel with previously purified Venus-hDHHC20 standards (8, 4, 1, 0.25 μM) from which a linear calibration curve could be constructed. Venus-hDHHC20 concentrations for most mutants were usually 2.5-6 μM. This translates to protein yields of ~7-16 μg per 100 mm cell culture dish. Of the mutants, AAxE and the ΔW267 mutants were the worst expressing. Therefore, in the case of these mutants, cells were transfected in 150 mm dishes (with 30 μg DNA) instead of 100 mm dishes. For the AAxE mutant, two such 150 mm dishes (equivalent to six 100 mm dishes) gave protein yields of ~14 μg. For the ΔW267 mutant, three 150 mm dishes (equivalent to nine 100 mm dishes) gave protein yields of ~3 μg. Coupled-enzyme assays to measure Michaelis-Menten constants were carried out as described above. For the acyl-CoA selectivity experiments, enzyme and acyl-CoA concentrations were kept fixed at 10 nM and 10 μM respectively. For mass spectrometry measurements, purified wild-type hDHHC20–2-BP sample was run on SDS-PAGE under reducing conditions. Gel was stained by coomassie brilliant blue, destained, and thoroughly washed in deionized water. Gel band (~5 μg protein) was excised, treated with the reducing agent TCEP and the free cysteines blocked with N-ethylmaleimide. Protein was digested with chymotrypsin and applied to a C4 reverse phase column run at a flow-rate of 300 nL/min on a Ultimate 3000 HPLC (Thermo-Dionex) connected to a Orbitrap Elite mass spectrometer (Thermo Scientific). DHHC20 cells tagged with N- or C-terminal Venus YFP were expressed in Cos7 cells using PEI transfection in 4-well or 8-well chambered microscope containers (Lab-Tek). Along with the Venus tagged DHHC20, we also co-expressed Cerulean CFP as a cytoplasmic marker and mCherry-TGN38 as a Golgi-marker with the mCherry RFP residing in the Golgi lumen. 24 hours post transfection, cells were washed with KHM (110 mM Potassium acetate, 20 mM HEPES, pH 7.2, and 3 mM MgCl2) buffer and then maintained in the same. Approximately a minute after initiation of time series imaging, cell plasma membrane was selectively permeabilized by the addition of digitonin dissolved in KHM to a final concentration of 22-25 μM. The permeabilization was followed by the disappearance of the cytoplasmic marker. Following disappearance of the cytoplasmic marker, trypsin prepared in KHM buffer was added to a final concentration of 1-2 mM and the cells further imaged for another minute or two until no change of any signal occurred. Control experiments were also done in which trypsin was added in the absence of digitonin. In these experiments, no significant loss in fluorescence was observed thus ruling out any pH or non-specific effects of trypsin on the cells. Imaging was carried out on a Zeiss LSM780 microscope equipped with Argon laser (458, 514 nm) and HeNe-laser (594 nm) with a Plan-Apochromat 63x/1.40 Oil objective and a 458/514/594 multiband filter set. Cerulean CFP, Venus YFP, and mCherry RFP were excited with the 458, 514, and 594 nm lasers respectively. Emission was collected from 462-506 nm, 516-587 nm, and 605-685 nm for Cerulean, Venus, and mCherry respectively. Images were analyzed using FIJI (64). Cos7 cells were seeded onto 8-well chamber plates (Lab-Tek) in 400 μl complete media and allowed them to grow and adhere to the surface of the plate for 16-18 hours. These cells were transfected with 100 ng of each vector GFP-zebrafish DHHC15, mCherry-TGN38, and BFP empty vector in serum free media with Lipofectamine 3000 transfection agent (Invitrogen). These transfected cells allowed to grow for 12-16 hours. The cells were permeabilized with 20 μM of digitonin in KHM buffer (110 mM potassium acetate, 2 mM Magnesium chloride and 50 mM HEPES, pH 7.5) for 30 s followed by trypsinization of the cells with 4 mM of trypsin in KHM buffer for 50 s during time dependent imaging experiment. Imaging data of adhered cells, from pre-permeabilized state to trypsinised state were acquired with confocal laser microscopy system equipped with a Plan Apochromat 63X/1.4NA oil immersion objective lens (LSM 780 Exciter, Carl Zeiss) at 37°C. Time-lapsed imaging was performed with the same instrument. Further, images were processed and fluorescence intensity was calculated using LSM and ImageJ softwares. PEG 300 and 2,5-Hexanediol were obtained from Sigma-Aldrich. PEG 400 was from USB. The buffers and salts were either from Sigma-Aldrich or Fisher Scientific. 2-bromopalmitate was from Sigma-Aldrich. All detergents were from Anatrace. All lipids and acyl coenzyme A was from Avanti Polar Lipids Inc. DTT and IPTG were from GoldBio. Digitonin (high purity) was from Calbiochem. Trypsin was obtained from Worthington Biochemicals. Methanol free 16% formaldehyde was obtained from Pierce-ThermoFisher and the DMAB complex from Sigma-Aldrich. Multiple sequence alignment was carried out using MUSCLE (65). Figure for sequence alignment with secondary structure was generated using ESPRIPT (66). www.sciencemag.org/content/359/6372/eaao6326/suppl/DC1 Figs. S1 to S12 Tables S1 to S8 References This is an article distributed under the terms of the Science Journals Default License. A lipid-binding cavity determines acyl chain length for a membrane-bound acyltransferase. A lipid-binding cavity determines acyl chain length for a membrane-bound acyltransferase. \",\n",
       "  'textLink': '/content/359/6372/eaao6326.full.txt',\n",
       "  'title': 'Fatty acyl recognition and transfer by an integral membrane S-acyltransferase',\n",
       "  'type': 'Research Article'},\n",
       " {'author': ['Teruko Danjo', 'Taro Toyoizumi', 'Shigeyoshi Fujisawa'],\n",
       "  'authorInstitution': ['Laboratory for Systems Neurophysiology, RIKEN Brain Science Institute, 2-1 Hirosawa, Wako, Saitama, 351-0198, Japan.',\n",
       "   'Laboratory for Neural Computation and Adaptation, RIKEN Brain Science Institute, 2-1 Hirosawa, Wako, Saitama, 351-0198, Japan.',\n",
       "   'Laboratory for Systems Neurophysiology, RIKEN Brain Science Institute, 2-1 Hirosawa, Wako, Saitama, 351-0198, Japan.'],\n",
       "  'category': 'Reports',\n",
       "  'date': '2018/01/12',\n",
       "  'doi': '10.1126/science.aao3898',\n",
       "  'imageLink': None,\n",
       "  'pdfLink': 'http://science.sciencemag.org/content/359/6372/213.full.pdf',\n",
       "  'pptLink': '/content/359/6372/213.full.txt',\n",
       "  'publisher': 'American Association for the Advancement of Science',\n",
       "  'text': 'Different sets of neurons encode the spatial position and orientation of an organism. However, social animals need to know the position of other individuals for social interactions, observational learning, and group navigation. Surprisingly, very little is known about how the position of other animals is represented in the brain. Danjo et al. and Omer et al. now report the discovery of a subgroup of neurons in hippocampal area CA1 that encodes the presence of conspecifics in rat and bat brains, respectively. Science, this issue p. 213, p. 218 An animal’s awareness of its location in space depends on the activity of place cells in the hippocampus. How the brain encodes the spatial position of others has not yet been identified. We investigated neuronal representations of other animals’ locations in the dorsal CA1 region of the hippocampus with an observational T-maze task in which one rat was required to observe another rat’s trajectory to successfully retrieve a reward. Information reflecting the spatial location of both the self and the other was jointly and discretely encoded by CA1 pyramidal cells in the observer rat. A subset of CA1 pyramidal cells exhibited spatial receptive fields that were identical for the self and the other. These findings demonstrate that hippocampal spatial representations include dimensions for both self and nonself. Spatial navigation requires the hippocampus (1, 2). The cognitive map theory states that spatial recognition in the hippocampus is allocentric (3–5). Place cells in the hippocampus are the physiological correlate of this representation because they are highly sensitive to changes in landmarks and contexts (6–13). The characterization of additional types of navigational representations, including head-direction cells and grid cells, has promoted our understanding of the neural mechanisms of allocentric spatial representations in the hippocampal-entorhinal cortex network (14–19). The studies of these neural maps have focused on the animal’s own position in space. It still remains unclear whether and how spatial information of nonself, such as the position of conspecifics, landmarks, and moving objects, is represented in the hippocampus. We designed an observational T-maze task using a pair of rats (hereafter termed “self” and “other”) and investigated how the other’s position is represented in the self’s hippocampus. The self was required to make a left/right choice to retrieve a reward based on the location of the other. We used two versions of the task, an “opposite-side rule” version in which the self rat had to choose the side opposite to the other’s location (Fig. 1A, fig. S1, and movie S1) and a “same-side rule” in which the self rat had to choose the same side as the other rat (fig. S1 and movie S2). During the T-maze task, neuronal activity in the self’s hippocampus (dorsal CA1) was recorded extracellularly (n = 3 pairs of rats; 88 ± 8.1% and 84 ± 11% correct performance with opposite-side and same-side rules, respectively). All analyses were performed on single units with pyramidal cell features and place fields in the task area (n = 1298 and 1205 units with opposite-side and same-side rules, respectively) (fig. S2) [see the supplementary materials (SM)]. (A) Schematic of the opposite-side rule. See fig. S1A. (B) Schematic of linearizing the trajectories on the T-maze. See figs. S3 and S4. (C to F) Units whose firing rates significantly depended on the other’s position. (Top) Other’s (left) and self’s (right) rate maps. (Second from top) Trajectory plots of other (left) and self (right). Gray lines, rats’ trajectories; red dots, spike positions. (Third from top) Linearized rate plots on other’s position (black lines). Red lines indicate statistically significant bands (P < 0.05) (see fig. S5). Blue thick line indicates the significant area. (Bottom) Joint rate maps. Other’s and self’s linearized positions are coordinated in horizontal and vertical axes, respectively. The maximum firing rate is indicated at the top of each map. (G to I) Linearized and normalized rate maps based on the other’s position (G), the self’s position (H), and the significant area on the other’s position (I), sorted by maximum rate positions on the other’s map (n = 1298 units). (J to L) Units with theta-phase precession based on the other’s position. (Top) Self’s rate maps (left), linearized rate plots, and theta phases of the spikes as a function of self’s position (right). (Bottom) Other’s rate maps (left), linearized rate plots, and theta phases of the spikes as a function of other’s position (right). (M) Correlation between theta phase and the other’s position. Color bars indicate significant correlation (P < 0.01; n = 59 units). We first examined how the location of the other rat was represented in the opposite-side rule observational task. Firing-rate maps of the self’s positions (“self’s rate map”) and the other’s positions (“other’s rate map,” which were obtained by replacing the self’s positional data with the other’s) revealed that in addition to the expected coding of the observer’s own position in space, the majority of units also displayed obvious place fields for the other (Fig. 1, C to F, top). To understand the positional relationship of the rat pair at the time of firing, we constructed joint rate maps by linearizing the rats’ trajectories and entering them into two-dimensional x-y axes (fig. S3). Here, the self’s and other’s place fields composed from identical spikes were combined into a single joint place field (Fig. 1, C to F, bottom, and fig. S4). We then examined whether these joint place fields were truly modulated by the other’s positions and not a mere consequence of the constraints in the positional relationships of the two rats. The null hypothesis was that the firing only depended on the self’s position and was independent of the other’s. By computing the surrogate firing rates that followed this null hypothesis (see the SM) (20), we identified areas with significantly higher firing rates for the other (P < 0.05) (Fig. 1, C to F, third from top, and fig. S5). A significant firing dependency on the other’s positions was detected for 85% of units (Fig. 1, G to I), indicating that the other’s spatial information was generally encoded by the place cells in this T-maze task. Importantly, the significant areas were not dependent on the self’s specific behavior or the other’s positions (Fig. 1, C to I). Moreover, the firing activities of most units were not dependent on the time periods in the trial (fig. S6). Similar results were also observed when we analyzed the same-side rule version of the task (fig. S7). In the hippocampus, spatial information is encoded not only by the average firing rates of the neurons but also by the timing of the spikes with respect to the phase of the underlying theta oscillations (7 to 11 Hz) (21–24). We next examined this temporal coding in the other’s spatial representations. To control for confounds of the motion of the observer, we focused on units whose rate peaks were located in the self’s starting positions in the opposite-side rule and analyzed the relationship between the other’s positions and the theta phases of the spikes (n = 78 units). Although the self was not running during the time course of this analysis, the local field potential was dominated by theta oscillations (fig. S8 and SM). In contrast to the lack of theta-phase precession relative to the self’s location, 75% of the units displayed significant theta-phase precession as a function of the other’s location (P < 0.01) (Fig. 1, J to M). The observed theta-phase precession to the other’s location was consistent with that to the self’s location, as reported previously (21–24), demonstrating that the other’s spatial information was also temporally encoded in the hippocampal pyramidal cells. We next sought to dissociate two types of units—those preferentially modulated by the other’s position (“other-preferred cells”) and those preferentially modulated by the self’s goal [“goal-preferred cells,” also previously described as “prospective cells” (25, 26)]. We took advantage of the two versions of our observational T-maze task (Fig. 2A). Other-preferred cells fire when the other is located on the same side of the side arms (left or right), irrespective of the rule during a given trial and, as a result, these neurons fire when the self’s goal is on a different side across the two conditions (Fig. 2B). This should further support the allocentric representation of the other’s place. In contrast, goal-preferred cells increase their firing rate only according to the future choice of the observer. By statistically comparing the other’s rate maps between the two versions, we defined other-preferred cells and goal-preferred cells based on these criteria (see the SM). We identified 58 other-preferred cells (Fig. 2, C to G, and fig. S9) and 252 goal-preferred cells (Fig. 2, H and I). Although the number of other-preferred cells was smaller than that of goal-preferred cells, it represented 13% of the place-responsive units examined in this analysis (Fig. 2F and SM), and the firing-rate peaks of both unit groups distributed all along the side arms in the other’s rate maps, constituting a full spatial map based on the position of the other rat (Fig. 2, G and I). Furthermore, the analysis of comparing the correct and the error trials in the opposite-side rule resulted in similar percentages of other-preferred and goal-preferred cells (fig. S10). (A) Schematic of the relationship of the other’s position and the self’s goal in opposite-side and same-side rules. (B) Schematic explaining other-preferred cells (left) and goal-preferred cells (right). Other-preferred cells fire preferentially when the other is on the same side, regardless of the rule. Similarly, goal-preferred cells fire when the self’s goal is on the same side. (C to E) Rate maps of other-preferred cells. (Top) Other’s rate maps in the opposite-side rule (left) and same-side rule (right). (Bottom) Self’s rate maps in the opposite-side rule (left) and the same-side rule (right). See fig. S9 for statistical methods. (F) Summary of other-preferred cells and goal-preferred cells (n = total 434 units). (G) Linearized and normalized rate maps of other-referred cells, sorted by peak rate positions. (H) Other’s (top) and self’s (bottom) rate maps of goal-preferred cells in the opposite-side rule (left) and the same-side rule (right). (I) Linearized and normalized other’s rate maps of goal-preferred cells in the opposite-side task (left) and same-side task (right), sorted by the maximum rate positions. We next looked for another type of spatial representation, “a common place field” for the self and other, for which a unit fires when either the other or the self is in a specific spatial location. We found 45 units that had common place fields in both the other’s and the self’s rate maps under the opposite-side rule (Fig. 3, A to E, and SM). We further tested the trajectory specificity of common place fields by separating left- and right-side targeting trials. Notably, we identified 51 of 1244 units in the opposite-side rule task that had trajectory-specific common place fields (Fig. 3, F and G; fig. S11; and SM), which was significantly higher than the number expected by chance (P < 0.05) (see SM). (A to C) Common place fields of representative units in the other’s (left) and the self’s (middle) rate maps. (Right) The products (element-wise multiplications) of normalized other’s and self’s firing rates in each pixel. (D) Linearized other’s (left) and self’s (middle) firing rates and their products (right). The products are the element-wise multiplications of the normalized firing rates of the self and the other at each position, of the units shown in (A) to (C). This value was used to identify units with common place fields. (E) Linearized and normalized rate maps of units with common place fields aligned by the position with the maximum value of the products (n = 45 of 908 units). (F and G) Trajectory-specific common place fields of two representative units. (Left) Other’s (top) and self’s (bottom) rate maps of whole trials. (Middle) Rate maps with the left and right trials separated. (Right) The products of normalized other’s and self’s firing rates of right (top) and left (bottom) targeting trials in each pixel. Arrows indicate the centers of common place fields. The maximum color value of the product map was set as 0.5, and the values in the product map indicate the maximum of the product. Because the allocentric representation of self’s place was also demonstrated by reliable reconstruction of self’s positions only from the spikes of place-cell ensembles (27, 28), we further tested whether the spikes of joint-place-cell ensembles could also reconstruct the other’s positions. First, we performed Bayesian decoding analysis with a leave-one-out strategy in each rule (SM). The reconstructed other’s trajectories revealed that the spikes of joint-place-cell ensembles contained sufficient information regarding the other’s positions (Fig. 4A and fig. S12A). Although the error distances between the actual and decoded positions were larger for the other’s decoding than those for the self, the differences in the error distances were as small as 5 cm per time window (200 ms) in both rules (other’s versus self’s error distances, 20.3 versus 16.1 cm in the opposite-side rule, 20.1 versus 15.1 cm in the same-side rule) (Fig. 4B). We then examined whether these spikes contained more information about the other’s place than that obtained by distributions of time spent in the positional relationships of the self and the other. We computed a control estimation of the other’s positions by reconstructing the self’s positions and then accordingly referring to the probability distributions of the positional relationships (fig. S12B and SM). For the Bayesian reconstructions in this analysis, the prior templates were computed from trials including both rules, and results were examined based on error distances in a block-wise manner (Fig. 4, C and D, and SM). Reconstructing trajectories without rule information apparently made the error distances of the other’s decoded positions larger, but they were significantly smaller than the control estimate of the other’s positions (Fig. 4D). The overall averages of the error distances of the other’s decoding were less than 40 cm and also were significantly smaller than those of the control estimation in both rules (Fig. 4E). (A) Reconstructed other’s and self’s positions in the opposite-side rule. Red and blue lines, reconstructed other’s and self’s positions, respectively. Black lines, actual trajectories of the other or self. (B) Mean error distances between actual and decoded positions (31 and 28 sessions from the opposite-side and same-side rules, respectively). (C to E) The reconstructions of other’s positions by using prior templates calculated from trials including both rules. (C) Schematic of the division of other’s trajectories into six blocks. (D) Error distances of reconstructed other’s positions in each block. Red bars, distance between the actual and decoded other’s positions. Gray bars, distance between the actual and estimated other’s positions (n = 22 sessions) (two-way repeated measures analysis of variance: P < 0.001, F5 = 83.95 for the opposite-side rule; P < 0.001, F5 = 72.6 for the same-side rule). (E) Averaged error distances per block (two-tailed paired t test: P < 0.001, t21 = –10.70 for the opposite-side rule; P < 0.001, t21 = –10.95 for the same-side rule). Data are mean ± SEM. We propose an extended model of hippocampal spatial representations that can include dimensions for both self and other (fig. S13). Our model, encompassing various types of spatial representations, can categorize spatial representations into four types: own place fields, joint place fields, other’s place fields, and common place fields (fig. S13, A to D). In particular, the common place field could be hypothesized to be a mirror representation of place (29, 30). Combinatorial representations of spatial information of self and nonself would open the door to examining whether this allocentric spatial representation extends more generally to other nonliving moving objects (31–33) and how it is generated in the hippocampal-entorhinal cortex network. Our data indicate that the place cells in the hippocampus encode sufficient spatial information for organizing the recognition of other animals, which is essential for social behavior (34, 35). www.sciencemag.org/content/359/6372/213/suppl/DC1 Materials and Methods Figs. S1 to S13 Movies S1 and S2 References (36–41) This is an article distributed under the terms of the Science Journals Default License. A group of neurons in hippocampal area CA1 of rats represents the position of conspecifics. A group of neurons in hippocampal area CA1 of rats represents the position of conspecifics. ',\n",
       "  'textLink': '/content/359/6372/213.full.txt',\n",
       "  'title': 'Spatial representations of self and other in the hippocampus',\n",
       "  'type': 'Research Article'},\n",
       " {'author': ['David B. Omer',\n",
       "   'Shir R. Maimon',\n",
       "   'Liora Las',\n",
       "   'Nachum Ulanovsky'],\n",
       "  'authorInstitution': ['Department of Neurobiology, Weizmann Institute of Science, Rehovot 76100, Israel.',\n",
       "   'Department of Neurobiology, Weizmann Institute of Science, Rehovot 76100, Israel.',\n",
       "   'Department of Neurobiology, Weizmann Institute of Science, Rehovot 76100, Israel.'],\n",
       "  'category': 'Reports',\n",
       "  'date': '2018/01/12',\n",
       "  'doi': '10.1126/science.aao3474',\n",
       "  'imageLink': None,\n",
       "  'pdfLink': 'http://science.sciencemag.org/content/359/6372/218.full.pdf',\n",
       "  'pptLink': '/content/359/6372/218.full.txt',\n",
       "  'publisher': 'American Association for the Advancement of Science',\n",
       "  'text': '↵* These authors contributed equally to this work. Different sets of neurons encode the spatial position and orientation of an organism. However, social animals need to know the position of other individuals for social interactions, observational learning, and group navigation. Surprisingly, very little is known about how the position of other animals is represented in the brain. Danjo et al. and Omer et al. now report the discovery of a subgroup of neurons in hippocampal area CA1 that encodes the presence of conspecifics in rat and bat brains, respectively. Science, this issue p. 213, p. 218 Social animals have to know the spatial positions of conspecifics. However, it is unknown how the position of others is represented in the brain. We designed a spatial observational-learning task, in which an observer bat mimicked a demonstrator bat while we recorded hippocampal dorsal-CA1 neurons from the observer bat. A neuronal subpopulation represented the position of the other bat, in allocentric coordinates. About half of these “social place-cells” represented also the observer’s own position—that is, were place cells. The representation of the demonstrator bat did not reflect self-movement or trajectory planning by the observer. Some neurons represented also the position of inanimate moving objects; however, their representation differed from the representation of the demonstrator bat. This suggests a role for hippocampal CA1 neurons in social-spatial cognition. It is important for social animals to know the spatial position of conspecifics, for purposes of social interactions, observational learning, and group navigation. Decades of research on the mammalian hippocampal formation has revealed a set of spatial neurons that represent self-position and orientation, including place cells (1–3), grid cells (4–6), head-direction cells (7–9), and border/boundary cells (10–12). However, it remains unknown how the location of other animals is represented in the brain. We designed an observational-learning task for Egyptian fruit bats (Rousettus aegyptiacus), which are highly social mammals that live in colonies with complex social structures (13). Bats were trained in pairs: In each trial, one bat (“observer”) had to remain stationary on a “start ball” and to observe and remember the flight trajectory of the other bat (“demonstrator”), which was flying roughly randomly to one of two landing balls (Fig. 1A,“demonstrator flying” in trials i and j). After a delay, the observer bat had to imitate the demonstrator bat and fly to the same landing ball to receive a reward (Fig. 1A, “observer flying,” and movies S1 and S2). This task had two key features: First, it required the observer to pay close attention to the demonstrator’s position and to hold this position in memory during the delay period (the average delay between the demonstrator’s return to the start ball and the observer’s takeoff was rather long: 12.7 ± 8.6 s; mean ± SD). Second, because the observer was stationary during the demonstrator’s flight, it allowed temporal dissociation between the effects of self-flights versus the flights of the other bat. (A) Behavioral setup inside a flight room (2.30 by 2.69 by 2.56 m). The demonstrator bat (blue) was trained to fly from the start ball, roughly randomly to either ball A or ball B, and back, and the observer bat had to imitate this flight. Two different trials are shown, one to ball A (trial i) and one to ball B (trial j). (B) Coronal section through dorsal hippocampus of one observer bat. Arrowhead, electrolytic lesion at end of tetrode track. (C) Five example cells (top view). For each cell, the left column shows the place-cell representation, based on spikes from the observer’s neuron and the self-flight-trajectories of the observer (Self), and the right column shows the social place-cell representation, based on spikes from the same observer’s neuron, plotted together with the demonstrator’s trajectories (Demo.). (Top) Flight trajectories (gray) with spikes overlaid (red). A, B, and S are landing balls A, B, and start ball; arrows denote flight-direction (↑, flying away from start-ball; ↓, flying toward the start ball). (Bottom) Firing-rate maps. Color scale ranges from zero (blue) to maximal firing rate (red; value indicated). The four leftmost cells are social place-cells; some of these neurons are also place-cells (cells 358, 254, and 52), and some are not (cell 266). Cell 313 is a “pure” place cell. Correlations between the firing-rate maps for self and other are indicated for each cell (correlations are undefined for cells 266 and 313 because one of the maps is flat). Scale bar, 50 cm. (D) Total number of significant social place-cells versus significant classical place cells that we recorded. (E) Number of place cells and social place-cells that were significantly tuned to one flight-direction (↑), the other flight-direction (↓), or both directions (↑↓). Classical place cells are in red (n = 261), and social place-cells are in blue (n = 68). (F) Locations of peak firing for all the significant maps for place cells (red dots, n = 371 cells × directions), and social place-cells (blue dots, n = 76 cells × directions); cells that had significant tuning in both directions were depicted twice; hence, the counts here are larger than in (D). Dots were randomly jittered by up to ±5 cm (half bin) for display purposes. (G) Average peak firing rate for all the classical place cells (red, n = 371 cells × directions) and all the social place-cells (blue, n = 76 cells × directions). **P < 0.01. (H) (Top) Distributions of correlation coefficients between classical place cell maps and social place-cell maps for all the neurons that encoded significantly either self-position or conspecific position and had >20 spikes per map (left histogram) or >300 spikes per map (right histogram). Gray, the data; black lines, cell-shuffling distributions (14). (Bottom) Map correlations increased with firing rate. Error bars, mean ± SEM; gray bars, the data; open bars, cell-shuffling; number of cells × directions included in the four bars: n = 334, 218, 137, and 91; *P < 0.05; **P < 0.01; n.s., nonsignificant. While the bats performed the task, we recorded the activity of 378 single neurons in the dorsal hippocampal area CA1 of four observer bats, using a wireless electrophysiology system (Fig. 1B) (14). For each neuron, we computed two firing-rate maps: a “classical” map, based on the self-movement flight trajectories of the observer—the standard depiction for place cells (Fig. 1C, “Self,” left map for each neuron)—and a nonclassical map based on the spikes recorded from the observer’s neuron together with the demonstrator’s flight trajectories (Fig. 1C, “Demo,” right maps) (14). We focused our analysis on the two-dimensional horizontal projections because the bats’ flights were confined mostly to a narrow horizontal slab around the height of the landing balls (fig. S1). A subpopulation of hippocampal CA1 neurons encoded the position of the demonstrator-bat (Fig. 1C, cells 358, 254, 52, and 266—the right map in each example—and fig. S2). We termed these neurons “social place-cells.” We classified 68 of the 378 recorded CA1 neurons (18.0%) as significant social place-cells—significantly encoding the position of the other bat—based on spatial information (95th percentile in a shuffling analysis) (14). Using the same criteria, 261 of the 378 recorded neurons (69.0%) significantly encoded the self-position of the observer bat when it was flying and were thus classified as place cells (Fig. 1D). Of the 261 place cells, 14.9% were also social place-cells. Conversely, of the 68 social place-cells, 57.4% (39 neurons) were also place cells (Fig. 1, C—cells 358, 254, 52—and D), whereas the remaining 29 social place-cells (42.6%) were not place cells. Most of these neurons (16 of 29 cells; 55.2%) became completely inactive during self-flights, although they encoded the conspecific’s position on interleaved demonstrator flights (examples are provided in Fig. 1C, cell 266, and fig. S2, cells 229 and 60). This new type of social-spatial representation exhibited several features that were similar to the standard place cell representation: Both representations showed directional selectivity (Fig. 1E and fig. S3), and both place cells and social place-cells tiled space rather uniformly (Fig. 1F). However, we found also clear differences between the two representations: First, the firing rates of the social place-cells were significantly lower than for the classical place-cells (unpaired t test, P < 0.01) (Fig. 1G) [firing-rates of classical place cells were similar to our previous report from CA1 of flying bats (15)]. Second, in the 39 cells that encoded both self-position and conspecific position—were both place cells and social place-cells—we found a wide range of correlation values between the representations for self and other. Some neurons exhibited high similarity between their place field and social place field (“congruent cells,” with positive correlations) (fig. S2, cells 68 and 45), whereas in other neurons, the place field and social place field were dissimilar (“noncongruent cells,” with negative correlations) (Fig. 1C, cells 358 and 254, and fig. S2, cell 242). Overall, we found a continuum from noncongruent to congruent representations (Fig. 1H, top histograms), but we also found a slight overrepresentation of congruent cells among higher-firing neurons (Fig. 1H, bottom, gray bars, and top right histogram). These data suggest partial remapping between the hippocampal representations of self-position and conspecific-position, which can be interpreted as reflecting the contextual difference between observing a conspecific versus self-movement. Next, we sought to rule out the possibility that social place fields might result from the observer’s head movements during the demonstrator’s flights. We therefore recorded head acceleration and head azimuth using a nine-axis motion-sensor that was placed on the observer’s head (14). When the demonstrator bat was flying, the observer bat hardly moved its head: There was a lack of changes in head acceleration of the observer bat during the flights of the demonstrator bat (Fig. 2A, middle and bottom, gray areas). Consistent with this, in most of the demonstrator flights, the head azimuth of the observer changed by less than 20°, which is equivalent to a very small head movement of less than 6 mm (Fig. 2, B—black traces and rightmost y axis, in magenta—and C). Such small head movements did not modulate the firing of social place-cells outside the task (Fig. 2D). These bats have a wide visual field and no fovea (13) and hence did not need to move their head in order to track the demonstrator. However, in some of the demonstrator flights (35.4%), the observer bat did move its head more than 20° (the value of 20° corresponds roughly to ± 1 SD in azimuth) (Fig. 2, B, gray traces and C, gray vertical lines). These deviant flights might have potentially modulated the firing of the neurons. To rule out this possibility, we recomputed the social firing-rate maps after excluding the deviant flights and found that these maps were very similar to the original maps (Fig. 2, E, examples, and F, population analysis). (A) Two example traces of flight velocity and acceleration of the observer’s head. (Top) Flight velocities of the two bats. (Middle) Acceleration of observer’s head. g, Earth’s gravitational acceleration. (Bottom) Very low acceleration around the demonstrator’s flight (gray rectangle). (B) Change in head azimuth of the observer (ΔAz) as function of time from onset of demonstrator’s flight. Each example shows data from 1 recording day in one flight direction; these examples correspond to the three cells in (E). Black, demonstrator flights in which the observer’s head moved <1 SD (σ = 20.18°, which corresponds to <6 mm movement; right y axis) (14). Gray, demonstrator flights that included deviant head movements of the observer bat that exceeded ±σ. Numbers indicate proportion of deviant flights out of all the flights on this day. (C) Distribution of ΔAz of the observer’s head, pooled over all days with significant social place-cells where motion-sensor data were recorded (n = 18 days, n = 35,284 samples). Gray lines mark 1 SD (σ = 20.18 o), which was the threshold used in (B) to define deviant flights. (D) Mean firing rate of social place-cells outside the task, triggered on the peak velocity of observer’s head movements, for all the 1-s segments with small angular displacement <20° (n = 14,893 segments, pooled over all significant social place-cells with motion-sensor data; shaded area indicates mean ± SEM). (E) Three example cells, showing high correlation between social place field maps before (top) and after (bottom) removal of all the flights that included observer head-movements [At bottom, we removed all gray-colored flights in (B) and the corresponding spikes]. (F) Blue histogram, distribution of correlation coefficients between social place-cell maps with and without removal of flights with observer movements. Black line, cell-shuffling distribution. We included here all the significant social place-cells where motion-sensor data were recorded (n = 29 cells × directions). Shown are high correlations between maps with versus without removal of flights with observer movements (blue histogram); t test with unequal variances, compared with cell-shuffling control (black): P < 10−26. (G) Example of a SWR. (Top) Spectrogram of the SWR. (Middle) Raw LFP trace (1 to 400 Hz bandpass). Scale bars, 30 ms and 200 μV. (Bottom) Spikes from four simultaneously recorded neurons (red ticks). Same time scale in all panels. (H) (Top) Mean SWR waveform, averaged across all recording days with social place-cells (n = 46 days; n = 9,092 SWRs). (Bottom) SWR-triggered firing rate, averaged over all neurons recorded during days with social place-cells (n = 276 neurons; shaded area, mean ± SEM). (I) Two social place-cells (columns), showing high stability with versus without flights that included SWRs (top versus bottom). (J) Distribution of correlation coefficients between social place-cell maps and the same maps after removal of flights with SWRs. Blue histogram, data for all cells with >20 spikes per map that had SWRs during observer flights (n = 20 cells × directions). Black line, cell-shuffling distribution. t test with unequal variances, data compared with cell-shuffling control: P < 10−140. (K) Two social place-cells (columns), showing high stability in correct trials (top) versus incorrect trials (bottom). (L) Distribution of correlation coefficients between social place-cell maps computed by using correct trials versus incorrect trials. Blue histogram, data for all neurons with >20 spikes per map (n = 43 cells × directions). Black line, cell-shuffling distribution. t test with unequal variances, data compared with cell-shuffling control: P < 10−8. We included in this analysis only cells with >15 correct flights and >15 incorrect flights; n = 43 cells × directions. (M) (Left) Simulated spatial distribution of social place fields, assuming that they are generated by place cell sequences with a ratlike sequence-speed of 8 m/s (14). (Right) Same, using a sequence speed of 43 m/s, which is scaled up to the flight speed of the demonstrator bat (corresponding to 20 times the bat’s flight speed in our task). Blue circles and crosses denote cells with preferred direction ↑ and ↓, respectively. A second potential interpretation is that social place fields may reflect planning of the upcoming flight trajectory by the observer bat. To rule out this possibility, we conducted three analyses. (i) Trajectory planning by hippocampal cell assemblies has been linked to sharp-wave-ripples (SWRs) (16). We recorded the local field potential (LFP) in the observer bat, then detected SWRs (Fig. 2, G and H) and tested whether removing the observer flights that contained SWRs would affect social place fields (Fig. 2, I and J) (14). The removal of these flights hardly affected the social place field maps, as indicated by very high map-correlations (Fig. 2, I, examples, and J, population), suggesting that social place fields are not created by SWR-associated trajectory planning. (ii) Next, we analyzed the neuronal activity during correct versus incorrect trials because studies in rats showed that hippocampal cell-assembly activity is strongly correlated to choice behavior on correct/incorrect trials (17). We reasoned that if the firing of the neurons reflects planning, then there would be a difference between social place-cell maps computed by using correct versus incorrect trials (where “incorrect” means that the demonstrator’s flight was followed by an incorrect flight of the observer)—because before incorrect flights, the observer bat is likely planning to fly to the opposite landing ball from the demonstrator. However, we found high correlations between correct-trial maps and incorrect-trial maps (Fig. 2, K, examples, and L, population). (iii) Trajectory planning has been linked to hippocampal place cell sequences (16), and such sequences might potentially create the social place fields that we observed. However, this seems highly unlikely because place cell sequences play extremely rapidly—at a speed of 8 m/s in rats (18), which is ~20 times faster than the running speed of the animal (18)—and therefore, all the firing of the observer’s neurons would be spatially compressed in one location, such as immediately after the takeoff of the demonstrator (14). Indeed, simulations of place-cell sequences confirmed this: All the place fields in this simulation were spatially compressed near the takeoff balls (Fig. 2M, blue crosses and circles), unlike the experimentally observed uniform distribution of social place fields (Fig. 1F, right). Together, this argues that social place fields cannot be explained via trajectory planning by the observer bat. Moreover, if trajectory planning in the observer’s brain is somehow synchronized precisely to the timing and velocity of each of the demonstrator’s flights—which seems rather unlikely—then it constitutes an explicit spatial representation of the position of the other bat. Classical place cells in CA1 represent the animal’s self-position in a world coordinate-frame: “allocentric coordinates” (1). To test whether social place-cells also form an allocentric representation, we exploited the fact that although the bats did not move their head much during the demonstrator flights (Fig. 2, A to C), the head did point in different azimuthal directions across different flights (we focused here on the azimuthal angle because the observer bats mainly moved their head in azimuth) (Fig. 3, A and B, and fig. S4) (14). For each of the social place-cells, we computed the median head azimuth of the observer (Fig. 3B, red line) and then used this median to divide all the demonstrator’s flights into two halves, corresponding to the observer bat looking right versus looking left (Fig. 3C, top versus bottom, respectively). If social place-cells are allocentric, then we expect similar maps irrespective of the head azimuth of the observer. Indeed, maps computed during right-viewing and left-viewing were rather similar (Fig. 3, C, examples, and D, population), which is consistent with an allocentric representation. Further, there was no relation between the map correlation and the average head direction difference between looking right and looking left [correlation coefficient (r) = –0.12, P = 0.59; the head-direction differences spanned a broad range, from ΔAz = 30° to 102°] (Fig. 3E and fig. S4), which also is consistent with an allocentric representation. These neurons are thus fundamentally different from vectorial goal-direction cells in the bat hippocampus, which represent the direction to navigational goals in egocentric coordinates (19). (A and B) Dividing the demonstrator’s flight data based on the observer’s head direction during demonstrator’s flights. (A) Schematic drawing of directional notations of the bat’s head relative to the two landing balls. (B) Distribution of the azimuthal head directions of the observer during demonstrator flights; data from 1 recording day. The median head direction (6.8°) is plotted in red. Direction 0° is parallel to the east-west wall of the room. (C) Two cells showing stability of their social place fields between right-pointing head directions (top) and left-pointing head directions (bottom). (D) Blue histogram, distribution of the correlation coefficients between right-looking maps and left-looking maps (blue), plotted for all the social place-cells for which we recorded motion-sensor data and had >20 spikes per map (n = 24 cells × directions); t test with unequal variances, compared with cell-shuffling control (black): P < 10−4. Black line, cell-shuffling distribution, consisting of correlations between left-looking maps from cell i and right-looking maps from cell j across all the cell pairs where i ≠ j. (E) Scatter plot of the similarity between right-looking and left-looking maps (y axis), versus the difference between the means of the right-looking and left-looking angles (x axis). No correlation was found (r = –0.12, P = 0.59; shown is a large span of azimuthal head-direction angles). Last, we asked whether a flying conspecific is represented differently from inanimate moving objects. We conducted additional experiments in two of the four recorded bats. These experiments included three sessions (Fig. 4A). Session 1 was conducted as before (Fig. 1A). In session 2, we moved an object either to ball A or to ball B, and the observer bat had to imitate it; it was the same task as before, but with an object instead of a conspecific. We termed this object an “informative object” (Fig. 4A, session 2, and fig. S5B) (14). In session 3, the observer bat was trained to hang at a fixed position on the start ball and to do nothing, while we moved a different object, a “noninformative object” (in this session, the observer bat did not receive reward and hence did not fly) (Fig. 4A, session 3, and fig. S5C). Both objects were similar in size to a flying bat (fig. S5). Surprisingly, we found quite a few CA1 cells that encoded the position of inanimate moving objects (Fig. 4, B, four top examples, and C, population); to our knowledge, this is the first report that the position of moving objects is explicitly represented in the hippocampus [a previous study reported modulation of place cell firing by the movement of another object, but not an explicit spatial representation of that object (20)]. Some of the CA1 cells represented both the inanimate objects and the conspecific (Fig. 4B, cells 184, 169, and 361); some cells represented only the objects (Fig. 4B, cell 182); and some cells represented only the conspecific (Fig. 4B, cell 221 and C, population summary). There were some differences between the representations of the conspecific and the inanimate objects. First, there was a slight trend for a better encoding of space (higher spatial information) going from the demonstrator bat to the informative object and to the noninformative object (Wilcoxon rank-sum tests, informative object versus noninformative object, P < 0.05; demonstrator bat versus noninformative-object, P = 0.093; demonstrator bat versus informative-object, P = 0.824; Kruskal-Wallis test, P = 0.092) (Fig. 4D). Second, whereas the representation of the conspecific was directional—akin to the directionality exhibited by self place fields—the representation of both objects was nondirectional, that is, rather similar in both directions (Fig. 4, E, example, and F, population). Third, the representation of the demonstrator bat was significantly less similar to any of the object representations, as compared with the similarity between the two object representations (Fig. 4, B, cells 184,169, and 361; E, examples of similar firing for both objects; and G, population) (controls for spatial-coverage, velocity, and firing-rate are provided in fig. S6). Fourth, we found a significant difference in the functional-anatomical gradient between social place-cells and object place cells, along the proximodistal axis of CA1. Social place-cells were significantly more prevalent closer to the distal border of CA1 (log odds-ratio test: P < 10−3) (Fig. 4H, middle, and fig. S7) (14), unlike object place-cells, which did not exhibit a significant proximodistal gradient (log odds-ratio test: P = 0.17) (Fig. 4H, right). Social place-cells exhibited the opposite pattern from classical place cells, which—consistent with previous reports in rats (21)—were significantly more prevalent near the proximal border of CA1 (log odds-ratio test: P < 10−5) (Fig. 4H, left). However, this result (Fig. 4H) was obtained from a single animal (out of the two bats tested in all three sessions), in which we had a sufficient number of neurons and good proximodistal span of tetrodes (14); future studies will need to examine this in more detail. Together, these results suggest that the representation of the conspecific is rather different from the representation of inanimate objects, indicating that the spatial coding of the conspecific is not a simple sensory response driven by any sensory stimulus that moves through the social place field. Rather, these are context-dependent cognitive representations. (A) Illustration of the behavioral task that we conducted in two of the four recorded bats. (B) Five example cells, showing firing-rate maps for the position of the demonstrator (left column), the position of the informative object (middle), and the position of the noninformative object (right). Cells 184, 169, and 361 encoded both the conspecific bat and the objects; cell 182 encoded only the objects; and cell 221 encoded only the conspecific. (C) Venn diagram summarizing the numbers of cells that represented significantly the conspecific and the two objects in these two bats. (D) Spatial information rate for all the cells representing significantly the position of the demonstrator (blue), the informative object (green), and the noninformative object (gray). Horizontal lines show the median, boxes show the 25th to 75th percentiles, and vertical lines show the 10th to 90th percentiles. (E) A cell exhibiting a difference in its firing-rate maps between different flight directions of the demonstrator bat (left column), but showing no directionality for the two objects (middle and right columns); compare the top and bottom maps for the two objects (direction ↑ vs ↓). (F) Directionality: population summary. Shown are correlations of firing-rate maps between the two flight directions: for the self-representation, the demonstrator bat, and the informative and noninformative objects (data for all cells in which at least one flight direction exhibited a significant map, and both maps contained >50 spikes per map). The maps are much more directional (lower correlations) for the demonstrator than for the two objects; t test for the correlations between the two directions for demonstrator-bat versus the two pooled objects: *P < 0.05. (G) Correlations of firing-rate maps for demonstrator bat versus informative object (left), demonstrator bat versus noninformative object (middle), and informative object versus noninformative object (right). Correlations here were computed for all cells in which at least one of the two maps was significant, and only for maps with >50 spikes; t test of the object-object similarity versus the conspecific-object similarities: **P < 0.01 for both comparisons. To increase the robustness of comparisons between demonstrator and objects, (C), (D), (F), and (G) included only cells that met a strict criterion of >25 flights per map and >50 spikes per map. (H) Functional anatomy along the proximodistal axis of CA1, for one of the two bats tested with three sessions (14). Shown is the percentage of significant tuning, separately for proximal and distal tetrodes. (Left) Place cells (Self). (Middle) Social place-cells (Demonstrator). (Right) Object place cells (pooled over both objects). ***P < 10−3; ****P < 10−5. We found in this work a subpopulation of cells in bat dorsal CA1 that encode the position of conspecifics, in allocentric coordinates. This representation could not be explained by self-head-movements or by self-trajectory-planning. The responses to the conspecific were directional, which is in line with the directionality of classical place cells, but can also be interpreted through the social difference between an approaching and receding conspecific. Social place fields are unlikely to reflect either distance-coding (observer-demonstrator distance) or time-coding (time since demonstrator-takeoff) because in both cases, we would expect rather symmetric firing fields on flights to both ball A and ball B, whereas nearly all the social place-cells had a firing field on one side only. However, an alternative interpretation is that these neurons encode a position-by-time signal: Namely, they encode the spatial side to which the demonstrator bat is flying, together with its time from takeoff. We also found qualitative differences between the spatial representations of conspecifics versus inanimate moving objects. The different encoding of conspecifics versus objects may arise from (partially) different mechanisms. For example, spatial representation of moving objects in CA1 might arise from convergence of spatial inputs from grid cells in the medial entorhinal cortex (4, 6) and object-related inputs from neurons in the lateral entorhinal cortex (22, 23); by contrast, social place-cells may also involve socially modulated inputs from CA2 (24, 25). Future studies are thus needed in order to search for social place-cells in the bat CA2, medial, and lateral entorhinal cortices, as well as in the ventral CA1, which was recently shown to be important for social memory (26). It may seem surprising that social place-cells were not discovered previously in several studies of rat hippocampus that looked for a modulation of classical place fields by the presence of conspecifics (27–29). We believe that the key difference is in the task: In those previous studies, there was no incentive for the animal to pay attention to the position of the conspecific; our task, in contrast, required the bat to pay close attention to the position of the other bat and to hold this position in memory during a 12.7-s average delay, which revealed a spatial representation for the other. This interpretation is consistent with many studies that showed that hippocampal representations are highly task-dependent, plastic, and memory-dependent (30–32). Additionally, this task created a high level of social interactions between the two bats: When the bats were together at the start ball, they often approached and touched each other and emitted many social vocalizations (fig. S8), and this intensely social situation may have contributed to the representation of the conspecific. There is an apparent similarity between the social place-cells, which encode the position of the other, and “mirror neurons” in monkeys, which encode the actions of the other (33). One difference, however, is that noncongruent social place-cells (Fig. 1C, cells 358 and 254) are still useful functionally because they encode meaningful information about the position of the other, whereas it is less clear how noncongruent mirror neurons in monkeys might be useful for the proposed functions of mirror neurons. Thus, social place-cells are conceptually different from mirror neurons, although both might possibly share a similar functional principle, whereby the same neuronal circuit can be used for self-representation as well as for representing conspecifics. Last, we speculate that social place-cells may play a role in a wide range of social behaviors in many species—from group navigation and coordinated hunting to observational learning, social hierarchy, and courtship—and may be relevant also for the representation of nonconspecific animals—for example, for spatial encoding of predators and prey. These results open many questions for future studies: How are multiple animals represented in the brain? Is there a different representation for socially dominant versus subordinate animals, and for males versus females? These and many other questions await investigation in order to elucidate the neural basis of social-spatial cognition. www.sciencemag.org/content/359/6372/218/suppl/DC1 Materials and Methods Figs. S1 to S8 References (34–42) Movies S1 and S2 This is an article distributed under the terms of the Science Journals Default License. A subpopulation of bat hippocampal CA1 neurons represents the spatial position of another bat. A subpopulation of bat hippocampal CA1 neurons represents the spatial position of another bat. ',\n",
       "  'textLink': '/content/359/6372/218.full.txt',\n",
       "  'title': 'Social place-cells in the bat hippocampus',\n",
       "  'type': 'Research Article'},\n",
       " {'author': ['Henriette E. Autzen',\n",
       "   'Alexander G. Myasnikov',\n",
       "   'Melody G. Campbell',\n",
       "   'Daniel Asarnow',\n",
       "   'David Julius',\n",
       "   'Yifan Cheng'],\n",
       "  'authorInstitution': ['Department of Biochemistry and Biophysics, University of California, San Francisco, CA 94143, USA.',\n",
       "   'Department of Molecular Biology and Genetics, University of Aarhus, 8000 Aarhus, Denmark.',\n",
       "   'Department of Biochemistry and Biophysics, University of California, San Francisco, CA 94143, USA.',\n",
       "   'Department of Biochemistry and Biophysics, University of California, San Francisco, CA 94143, USA.',\n",
       "   'Department of Biochemistry and Biophysics, University of California, San Francisco, CA 94143, USA.',\n",
       "   'Department of Physiology, University of California, San Francisco, CA 94143, USA.',\n",
       "   'Department of Biochemistry and Biophysics, University of California, San Francisco, CA 94143, USA.',\n",
       "   'Howard Hughes Medical Institute, University of California, San Francisco, CA 94143, USA.'],\n",
       "  'category': 'Reports',\n",
       "  'date': '2018/01/12',\n",
       "  'doi': '10.1126/science.aar4510',\n",
       "  'imageLink': None,\n",
       "  'pdfLink': 'http://science.sciencemag.org/content/359/6372/228.full.pdf',\n",
       "  'pptLink': '/content/359/6372/228.full.txt',\n",
       "  'publisher': 'American Association for the Advancement of Science',\n",
       "  'text': 'Transient receptor potential melastatin (TRPM) ion channels constitute the largest TRP subfamily and are involved in many physiological processes. TRPM8 is the primary cold and menthol sensor, and TRPM4 is associated with cardiovascular disorders. Yin et al. and Autzen et al. shed light on the general architecture of the TRPM subfamily by solving the structures of TRPM8 and TRPM4, respectively (see the Perspective by Bae et al.). The three-layered architecture of the TRPM8 channel provides the framework for understanding the mechanisms of cold and menthol sensing. The two distinct closed states of TRPM4, with and without calcium, reveal a calcium-binding site and calcium-binding-induced conformational changes. Science, this issue p. 237, p. 228; see also p. 160 Transient receptor potential (TRP) melastatin 4 (TRPM4) is a widely expressed cation channel associated with a variety of cardiovascular disorders. TRPM4 is activated by increased intracellular calcium in a voltage-dependent manner but, unlike many other TRP channels, is permeable to monovalent cations only. Here we present two structures of full-length human TRPM4 embedded in lipid nanodiscs at ~3-angstrom resolution, as determined by single-particle cryo–electron microscopy. These structures, with and without calcium bound, reveal a general architecture for this major subfamily of TRP channels and a well-defined calcium-binding site within the intracellular side of the S1-S4 domain. The structures correspond to two distinct closed states. Calcium binding induces conformational changes that likely prime the channel for voltage-dependent opening. Transient receptor potential (TRP) channels comprise an extended superfamily of membrane ion channels that mediate diverse cellular and physiological functions. Permeability to both mono- and divalent cations is a defining feature for the majority of TRP channels (1). However, the TRP melastatin subfamily member 4 (TRPM4) channel is impermeable to Ca2+ yet is activated by intracellular Ca2+ (2). Activation of TRPM4 depolarizes the plasma membrane through Na+ entry, which in turn enhances Ca2+ influx through Ca2+-permeable channels or otherwise modulates Ca2+ oscillations (2, 3). Like many other TRP channel subtypes, TRPM4 displays voltage sensitivity, as evidenced by pronounced outward rectification of Ca2+-activated TRPM4 currents in response to voltage ramps (4). Binding of Ca2+ to TRPM4 is hypothesized to precede voltage-dependent opening, thereby giving rise to two closed states: Ca2+-unbound and Ca2+-bound (5). However, it is unclear where the Ca2+-binding site is located in the channel, and a structural basis for voltage sensitivity and ion selectivity remains elusive. Here we present two single-particle cryo–electron microscopy (cryo-EM) structures of the human TRPM4 channel in nanodiscs, in both Ca2+-bound and -unbound closed states. Comparison of the two structures reveals a Ca2+-binding site in the transmembrane domain, as well as conformational changes induced by Ca2+. Lipid molecules, including cholesteryl hemisuccinate (CHS), form tight interactions with the channel, likely stabilizing it in the lipid bilayer. Recombinant full-length human TRPM4 (hTRPM4b) was purified in detergent and reconstituted into lipid nanodiscs, as described for other TRP ion channels (6) (fig. S1). Three-dimensional reconstructions with C4 symmetry were obtained in the presence of either 5 mM EDTA or CaCl2 to overall resolutions of 3.2 and 3.1 Å, respectively (Fig. 1, A and B, and figs. S2, A to F, and S3, A to F). In both structures, the transmembrane domain and part of the soluble domain are well resolved. However, the soluble regions distal to the membrane and the symmetry axis are of lower resolution, indicating conformational flexibility (figs. S2G and S3G). Focused refinement of the transmembrane domain with the central coiled-coil improved the local resolution and enabled de novo atomic model building of these domains for both samples (figs. S2, F and H; S3, F and H; S4, B and D; and S5). For the rest of the cytoplasmic domains, further classification and focused refinement of individual monomers without symmetry improved the resolution to a level sufficient for de novo model building (Fig. 1, C to F and figs. S2, I to K; S3, I to K; S4, C and E; and S5). The complete data processing scheme is described in the supplementary methods and fig. S4. (A and B) Side (A) and top (B) views of unsharpened (transparent light blue) and sharpened cryo-EM density maps of hTRPM4 in nanodiscs, with each subunit colored differently. (C) Atomic model of the tetrameric hTRPM4 in ribbons, in the same orientation and colors as the density map in (A). (D) Atomic model of the hTRPM4b monomer in ribbons, with domains labeled. (E) Bottom view of the hTRPM4 structure, showing interactions between neighboring subunits. (F) Schematic representation of the major structural components in hTPPM4b. Dashed lines denote regions where density was insufficient for model building. Each domain is labeled and color coded to match the domain representation in (D). The transmembrane core contains six helices (S1 to S6) arranged in a domain-swapped architecture (Fig. 1, C and D) reminiscent of many tetrameric voltage-gated cation channels and all other TRP channels to date (7–13). hTRPM4 has additional membrane-embedded fragments that surround the S1-S4 domain at the inner leaflet of the membrane (Fig. 1, D and F), a feature that might be distinct to the TRPM subfamily. The C terminus of each subunit joins to form a parallel coiled-coil resembling that of the TRPA1 channel (11). The coiled-coil is surrounded by a large, intertwined cytoplasmic domain composed of four N-terminal TRPM homology regions (MHR1 to MHR4), which are highly conserved within the TRPM subfamily (14). MHR1 and MHR2 contain an eight-stranded β sheet surrounded by eight α helices. MHR3 and MHR4 are composed of stacked α helices and are linked to the transmembrane domain through MHR4, which clasps the TRP domain, thereby forming an interaction between the cytoplasmic domain and the transmembrane core (Fig. 1D). Between neighboring monomers, MHR1 of one subunit interacts with MHR3 of the adjacent subunit to form a ring that does not interact with the central coiled-coil (Fig. 1E). Comparison of Ca2+-free and Ca2+-bound structures reveals an extra density in a hydrophilic pocket within the cytoplasmic side of the S1-S4 domain (Fig. 2A and fig. S6). Several lines of analysis suggest that this density represents a bona fide bound Ca2+ ion: (i) Direct comparison between both half maps and the combined maps of the CaCl2 and EDTA samples confirms the existence and precise location of the additional density in the CaCl2 map (fig. S6, A to C and E to G). (ii) The omit densities calculated as the difference between the experimental maps and a map calculated from the refined atomic models without Ca2+ show a strong density corresponding to a bound Ca2+ ion (fig. S6D) in the CaCl2 structure, but not in the EDTA structure. (iii) The difference map between the CaCl2 and EDTA maps reveals a clear density with a high signal-to-noise ratio (Fig. 2A and fig. S6H). (iv) Finally, side chain densities of surrounding negatively charged residues are stronger in the CaCl2 structure than in the EDTA structure (fig. S6, A and E), consistent with the observation that negatively charged side chains are generally weaker in a cryo-EM density map unless they are engaged in specific interactions (15). (A) Coordination of the bound Ca2+ ion by Glu828, Gln831, Asn865, and Asp868, with distances labeled. The density of the CaCl2 structure (gray mesh) is contoured at σ = 5 and overlaid with the difference density (blue mesh) between the CaCl2 structure and the EDTA structure, contoured at σ = 14. (B) Ca2+-binding site within the S1-S4 domain in the absence of a bound ion. (C) The same site with a bound ion. Side chains are labeled. The bound Ca2+ ion is shown in magenta. Single-letter abbreviations for the amino acid residues are as follows: A, Ala; C, Cys; D, Asp; E, Glu; F, Phe; G, Gly; H, His; I, Ile; K, Lys; L, Leu; M, Met; N, Asn; P, Pro; Q, Gln; R, Arg; S, Ser; T, Thr; V, Val; W, Trp; and Y, Tyr. The Ca2+ ion is directly coordinated by side chains of Glu828 and Gln831 from S2 and Asn865 from S3 (Fig. 2A). Asp868 is also near the ion and may participate in its coordination. Together, these side chains contribute at least four oxygen atoms that can coordinate the Ca2+ ion. The Ca2+-binding site is located within a hydrophilic pocket that is large enough to accommodate additional water molecules for further coordination of the bound ion. This pocket is connected to the cytoplasmic space through a narrow path between the TRP domain and the S2-S3 linker (fig. S6, I and J). Within the pocket, Arg905 from a single 310 helical turn in S4 and Tyr790 from S1 are located above the Ca2+-binding site. In the absence of Ca2+, Arg905 is coordinated by Glu828 and Asp868 (Fig. 2B). Upon Ca2+ binding, the S2-S3 linker translates ~1.5 Å such that Tyr859 supports a tighter conformation for coordination of Ca2+ by Glu828, Gln831, Asn865, and Asp868 (fig. S6L). Furthermore, the side chain of His908 in S4 transitions from a π-π stacking arrangement with Trp864 in S3 to form a new interaction with Cys867 (Fig. 2, B and C, and fig. S6, K to N). Without coordination of Glu828 and with rearrangement in S3 and S4, Arg905 moves slightly upward (fig. S6N). The configuration of Arg905 and Tyr790 is reminiscent of a positive gating charge (arginine) and a charge-transfer center (tyrosine) observed in voltage-gated potassium channels (16). We speculate that Ca2+ binding moves Arg905 up toward Tyr790 to prime the channel for voltage-dependent opening. A previous mutagenesis study showed that the Glu1068→Gln1068 mutation significantly reduces TRPM4 Ca2+ sensitivity (17). Glu1068 is located in the pathway leading to the Ca2+-binding site from the cytoplasmic space (fig. S6, I and J), and this mutation likely reduces the Ca2+ accessibility to the site. Of the four residues that constitute the Ca2+-binding site, Asn865 and the negative charge of Asp868 are conserved throughout the TRPM subfamily, whereas Glu828 and Gln831 as well as Tyr859 and Glu1068 are conserved only in subfamily members shown to be Ca2+ dependent: TRPM2, TRPM4, TRPM5, and TRPM8 (18–20) (fig. S7). Therefore, an analogous binding site is likely present in these channels. Electrophysiological and mutagenesis studies of TRPM8 suggested that Ca2+ binds directly to an analogous site in that channel to confer sensitivity to icilin, a synthetic cooling agent. Specifically, substituting the residues corresponding to Asn865 and Asp868 results in icilin-insensitive channels that retain robust Ca2+-independent responses to cold and menthol (21). The central pore of hTRPM4 is formed by S5 and S6 and the intervening re-entrant pore helix and pore loop. Together, these elements form an ion permeation pathway with two restriction sites, similar to other TRP channels (Fig. 3, A and B). As seen in TRPV1 (10), the outer pore is shaped as a funnel, with a negatively charged residue facing the funnel, attracting cations (fig. S8, A and B). Two highly conserved residues, Phe975 and Gly976, located in the bottom of the pore loop, form the narrowest restriction point of the upper pore. The pore diameter at this site is very similar to that of the open gate in TRPV1 (22) (Fig. 3B), which is sufficient to accommodate partially dehydrated monovalent cations. However, no density corresponding to bona fide coordinated ions is found within the pore (fig. S8, C and D). The location of a previously identified putative selectivity filter (981EDMDVA986) (23) does not coincide with this restriction site but is located further toward the extracellular face (Fig. 3C). At the bottom of the ion permeation pathway, side chains of opposing Ile1040 residues in S6 form a tight seal at the lower restriction site, signifying that both structures are in closed conformations (Fig. 3, A and B). (A) The solvent-accessible pathway along the ion permeation pore represented by dots between two opposing monomers shown in yellow and blue ribbons. Residues aligning the pathway are shown in sticks and labeled. (B) The pore radius of the hTRM4 EDTA (black) and CaCl2 (blue) structures overlaid with the pore radius of TRPV1 in its closed (purple) and open (orange) states. (C) Close-up view of the pore helix and pore loop. The two single-turn π helices are depicted in red in the middle of S6 and the pore helix. The putative selectivity filter is shown in orange. The inset represents the density of a bound lipid, fitted with the atomic model of CHS, which forms a tripartite complex with S1 from one monomer and the pore helix of the neighboring monomer. The pore helix of hTRPM4 has an additional turn compared with that in other TRP channels (Fig. 3C) (9–13). The middle of the pore helix features a single-turn π helix (Arg964-Arg969) followed by a proline residue, Pro970, conserved in all human TRPM channels except TRPM2. Likewise, similar to what was first identified in TRPV1 (10) and subsequently observed in other TRP channels (7, 11), there is a one-turn π helix (Val1030-Leu1035) in the middle of the S6 helix (Fig. 3C). Although the exact role of these single-turn π helices is unclear, they could potentially facilitate helix bending under different functional states, leading to movement of the lower gate or modulation of the upper pore. Outside of the pore, hTRPM4 has an extensive extracellular loop comprising more than 30 residues that connect the pore loop to the S6 helix. The loop is clearly resolved, likely stabilized by a disulfide bond between two cysteine residues (Cys993 and Cys1011) (Fig. 3C and fig. S8, F and G), a feature predicted to exist in the majority of TRPM channels. This disulfide bond is not seen in the CaCl2 structure (fig. S7H), where radiation damage may have caused disulfide bond breakage (24). The loop also contains a glycosylation site (Asn992) with the attached glycan pointing toward the extracellular space (Fig. 3C and fig. S7, G and H). In addition to the transmembrane S1-S6 domain, hTRPM4 has distinct membrane-embedded α-helical segments (Fig. 4A). These segments surround the exterior of the S1-S4 domain within the inner leaflet of the membrane and mediate extensive interactions with the soluble domain. Preceding the S1 helix, there are two short helices shaped as an inverted “V” embedded within the inner leaflet of the membrane (Fig. 4B). We term this the “pre-S1 elbow,” similar to that observed in the mechanotransduction channel NOMPC (9). A disordered loop, unresolved in both structures, connects the pre-S1 elbow to an amphipathic helix, which we term the “pre-S1 shoulder,” positioned at the inner surface of the membrane (Fig. 4C). The pre-S1 shoulder helix contains large hydrophobic residues buried in the membrane and a number of charged residues facing the cytosol. Among these charged residues, Arg767 was identified as important for interactions with the phosphatidylinositol lipids PIP2 and PIP3 (25). (A) Side view of the hTRPM4 monomer, with membrane-embedded helical segments labeled and in solid color. (B) The pre-S1 elbow (gold) and pre-S1 shoulder (yellow). Lipid density between the pre-S1 elbow and S1 is modeled with CHS. Side chains interacting with CHS are shown as sticks. (C) Same as (B), viewed from the opposite direction. (D) Cytoplasmic domain MHR3 interacts with the S2-S3 linker. (E) The CH1 helix after the TRP domain. Charged residues in the pre-S1 shoulder (C) and CH1 (E) are shown in sticks. Missing links are shown in dashed lines in (A) and (D). An extended S2-S3 linker distinct to the TRPM subfamily forms a short amphipathic helix, positioned at the membrane surface, near the Ca2+-binding site (Fig. 2B). The loop that connects S2 with the S2-S3 linker is ~12 residues longer in hTRPM4 compared with other human TRPM channels and extends beyond the membrane bilayer, interacting with MHR4. Ser839 in the linker is a predicted phosphorylation site important for trafficking TRPM4 to the plasma membrane (26); however, this linker is only weakly resolved in the CaCl2 structure. In addition, the S2-S3 linker also interacts loosely with the cytoplasmic domain MHR3 (Fig. 4D). The last segment is positioned between the pre-S1 shoulder and the S2-S3 linker helix (Fig. 4, A and F). Here, a loop fills the gap between S1 and S2 and connects the conserved TRP domain to the C-terminal helix 1 (CH1), which is located at a steep angle relative to S1, extending downward from the middle of the membrane bilayer to the cytoplasmic surface more than 20 Å away from the end of the TRP domain (Fig. 4E). The C terminus of CH1 contains five arginine residues, similar to the pre-S1 shoulder mentioned above, which likely constitute interaction partners for negatively charged lipid headgroups at the membrane interface. CH1 connects to CH2 through a 19-residue linker, which was only partially resolved in the structures and was observed to associate with MHR4 and the C-terminal end of the TRP domain. A number of lipids were observed in our structures (fig. S9A). In addition to annular phospholipids, we identified three densities as CHS on the basis of its characteristic shape (Figs. 3C and 4B and fig. S9B). CHS is often used as a cholesterol analog (27); thus, a cholesterol molecule is likely to bind at the same sites. One CHS molecule fills a cleft at the back side of the pore, interacting with the S6 helix and the pore loop or the proposed selectivity filter of one subunit and the pore helix of the neighboring subunit. This arrangement, together with the relative rigidity of CHS over annular phospholipids, may stabilize the conformation of the pore (Fig. 3C and fig. S8E). A second CHS molecule is positioned at a location equivalent to the vanilloid binding pocket in TRPV1 (6) (fig. S9C). A third CHS is located at the pre-S1 elbow, which together with S1 and S4 creates a cavity (Fig. 4B). Although there is no noticeable change of the pore between the two structures (Fig. 3B and fig. S10, A and C), we observe subtle yet well-defined changes in the region surrounding the Ca2+-binding site (fig. S10). In addition, CH2 is horizontally displaced by ~2.5 Å around the central coiled-coil, measured from the Cα of Glu1116, in the distal end of CH2 (fig. S10D). The rotation results in a slight tightening of the central coiled-coil (fig. S10E), likely stabilizing it, as evident by the observation that more residues in the C-terminal end of the coiled-coil were resolved in the CaCl2 structure (figs. S2G and 3G). Given that the lower gate remains closed in both structures, such conformational changes are insufficient for channel opening. However, as the extensive soluble domains are implicated in interactions with various cofactors (28), such conformational changes may be important functional features that enable the channel to better detect or responds to its cofactors (28). www.sciencemag.org/content/359/6372/228/suppl/DC1 Materials and Methods Figs. S1 to S10 Tables S1 to S3 References (29–51) This is an article distributed under the terms of the Science Journals Default License. Structures of a human cation channel revealed by single-particle cryo–electron microscopy elucidate the calcium-binding site. Structures of a human cation channel revealed by single-particle cryo–electron microscopy elucidate the calcium-binding site. ',\n",
       "  'textLink': '/content/359/6372/228.full.txt',\n",
       "  'title': 'Structure of the human TRPM4 ion channel in a lipid nanodisc',\n",
       "  'type': 'Research Article'}]"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "d['2018']['Jan 12']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 68   "
     ]
    }
   ],
   "source": [
    "authors = []\n",
    "links = []\n",
    "count = 0\n",
    "for p in d:\n",
    "    for i in d[p]:\n",
    "        for j in d[p][i]:\n",
    "            if len(j['author']) >20 :\n",
    "                count+=1\n",
    "                if count <= 61:\n",
    "                    continue\n",
    "                urlretrieve(j['pdfLink'],'../pdf/%s.pdf'%j['title'].replace('/',' '))\n",
    "                print('\\r %s   '%str(count),end='')\n",
    "                time.sleep(60*(1+random.random()))\n",
    "                "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "128"
      ]
     },
     "execution_count": 38,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(links)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {
    "collapsed": false,
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "http://science.sciencemag.org/content/327/5964/425.full.pdf\n"
     ]
    }
   ],
   "source": [
    "print(links[0])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "from urllib.request import urlretrieve\n",
    "import time\n",
    "import random"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "ename": "IsADirectoryError",
     "evalue": "[Errno 21] Is a directory: '../pdf'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mIsADirectoryError\u001b[0m                      Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-41-050877fe84ee>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0mi\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;36m0\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mlink\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mlinks\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 3\u001b[0;31m     \u001b[0murlretrieve\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mlink\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m'../pdf'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      4\u001b[0m     \u001b[0mi\u001b[0m\u001b[0;34m+=\u001b[0m\u001b[0;36m1\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      5\u001b[0m     \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'/r %s   '\u001b[0m\u001b[0;34m%\u001b[0m\u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mi\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/Users/wangyuhan/anaconda/lib/python3.6/urllib/request.py\u001b[0m in \u001b[0;36murlretrieve\u001b[0;34m(url, filename, reporthook, data)\u001b[0m\n\u001b[1;32m    256\u001b[0m         \u001b[0;31m# Handle temporary file setup.\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    257\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mfilename\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 258\u001b[0;31m             \u001b[0mtfp\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mopen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mfilename\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m'wb'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    259\u001b[0m         \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    260\u001b[0m             \u001b[0mtfp\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mtempfile\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mNamedTemporaryFile\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdelete\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mFalse\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mIsADirectoryError\u001b[0m: [Errno 21] Is a directory: '../pdf'"
     ]
    }
   ],
   "source": [
    "i = 0\n",
    "for link in links:\n",
    "    urlretrieve(link,'../pdf')\n",
    "    i+=1\n",
    "    print('/r %s   '%str(i))\n",
    "    time.sleep(10*(1+random.random()))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'sd f'"
      ]
     },
     "execution_count": 48,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "'sd/f'.replace('/',' ')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
